2020-03-19 19:42:53 INFO   MockWebServer[50009] starting to accept connections 
2020-03-19 19:42:54 INFO   Initialising Kubernetes support 
2020-03-19 19:42:54 INFO   MockWebServer[50009] received request: GET /api/v1/namespaces/test/pods/b-cn1002.hpc2n.umu.se HTTP/1.1 and responded: HTTP/1.1 200 OK 
2020-03-19 19:42:54 INFO   Kubernetes support initialised.
	Pod name: b-cn1002.hpc2n.umu.se
	app label: zalenium
	zalenium service name: zalenium
	Selenium Pod Resource Limits: {memory=Quantity(amount=4Gi, format=null, additionalProperties={}), cpu=Quantity(amount=500m, format=null, additionalProperties={})}
	Selenium Pod Resource Requests: {memory=Quantity(amount=1Gi, format=null, additionalProperties={}), cpu=Quantity(amount=250m, format=null, additionalProperties={})} 
2020-03-19 19:42:55 INFO   Initialising Kubernetes support 
2020-03-19 19:42:55 INFO   MockWebServer[34965] starting to accept connections 
2020-03-19 19:42:55 INFO   MockWebServer[34965] received request: GET /api/v1/namespaces/test/pods/b-cn1002.hpc2n.umu.se HTTP/1.1 and responded: HTTP/1.1 200 OK 
2020-03-19 19:42:55 INFO   Kubernetes support initialised.
	Pod name: b-cn1002.hpc2n.umu.se
	app label: zalenium
	zalenium service name: zalenium
	Selenium Pod Resource Limits: {memory=Quantity(amount=4Gi, format=null, additionalProperties={}), cpu=Quantity(amount=500m, format=null, additionalProperties={})}
	Selenium Pod Resource Requests: {memory=Quantity(amount=1Gi, format=null, additionalProperties={}), cpu=Quantity(amount=250m, format=null, additionalProperties={})} 
2020-03-19 19:42:56 INFO   [BS] Account max. concurrency was NOT fetched from https://www.browserstack.com/automate/plan.json 
2020-03-19 19:42:56 INFO   Registered a node http://hub-cloud.browserstack.com:80 
2020-03-19 19:42:56 INFO   Registered a node http://localhost:30000 
2020-03-19 19:42:56 INFO   [DS]  Setting up 0 nodes... 
2020-03-19 19:42:56 INFO   0 containers were created, it will take a bit more until all get registered. 
2020-03-19 19:42:56 INFO   Test will be forwarded to BrowserStack, {browserName=internet explorer, platformName=WIN8} 
2020-03-19 19:42:56 INFO   Currently using 0 paralell sessions towards BrowserStack. Attempt to start one more. 
2020-03-19 19:42:56 INFO   Trying to create a new session on test slot {browserName=BrowserStack, maxInstances=1, platformName=ANY} 
2020-03-19 19:42:57 WARNING Tried to release session with internal key[INFO] Tests run: 6, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 5.6 s - in de.zalando.ep.zalenium.proxy.BrowserStackRemoteProxyTest
[INFO] Running de.zalando.ep.zalenium.proxy.DockerSeleniumRemoteProxyTest
2020-03-19 19:43:24 [main] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:43:25 [main] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:43:25 [main] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:43:25 [main] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:43:25 [main] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:43:25 [main] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:43:26 [main] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:43:27 [main] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:43:27 [main] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:43:28 [main] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:43:29 [main] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:43:30 [main] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:43:30 [main] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:43:32 [main] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:43:32 [main] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:43:33 [main] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:43:33 [main] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:43:33 [main] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:43:33 [main] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:43:33 [main] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:43:33 [main] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:43:34 [main] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:43:35 [main] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:43:35 [main] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:43:36 [main] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:43:37 [main] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:43:38 [main] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:43:39 [DockerSeleniumNodePoller container [b-cn1002.hpc2n.umu.se]] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:43:39 [DockerSeleniumNodePoller container [b-cn1002.hpc2n.umu.se]] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:43:39 [main] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:43:39 [DockerSeleniumNodePoller container [b-cn1002.hpc2n.umu.se]] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:43:39 [main] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:43:39 [DockerSeleniumNodePoller container [b-cn1002.hpc2n.umu.se]] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:43:39 [main] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:43:39 [DockerSeleniumNodePoller container [b-cn1002.hpc2n.umu.se]] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:43:39 [main] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:43:39 [DockerSeleniumNodePoller container [b-cn1002.hpc2n.umu.se]] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:43:39 [main] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:43:39 [main] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:43:39 [main] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:43:40 [main] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:43:40 [main] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
[INFO] Tests run: 57, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 40.431 s - in de.zalando.ep.zalenium.proxy.DockerSeleniumRemoteProxyTest
[INFO] Running de.zalando.ep.zalenium.proxy.DockerSeleniumStarterRemoteProxyTest
2020-03-19 19:44:00 [main] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:44:01 [main] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:44:01 [main] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:44:02 [main] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:44:03 [main] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:44:03 [main] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:44:04 [main] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:44:05 [main] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:44:05 [main] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:44:06 [main] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:44:07 [main] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:44:07 [main] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:44:08 [main] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:44:09 [main] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:44:09 [main] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:44:10 [main] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:44:11 [main] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
2020-03-19 19:44:12 [main] WARN  io.fabric8.kubernetes.client.Config  - Error reading service account token from: [/var/run/secrets/kubernetes.io/serviceaccount/token]. Ignoring.
[INFO] Tests run: 54, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 31.512 s - in de.zalando.ep.zalenium.proxy.DockerSeleniumStarterRemoteProxyTest
[INFO] Running de.zalando.ep.zalenium.proxy.TestingBotRemoteProxyTest
2020-03-19 19:44:14 [jersey-client-async-executor-3] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-3] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-3] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-3] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-3] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-3] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-17] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-17] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-17] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-17] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-17] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-17] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-4] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-4] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-4] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-4] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-4] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-4] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-10] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-10] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-10] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-10] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-10] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-10] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-5] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-5] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-5] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-5] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-5] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-5] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-2] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-2] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-2] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-2] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-2] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-2] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-1] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-1] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-1] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-1] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-1] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-1] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-7] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-7] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-7] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-7] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-7] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-7] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-8] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-8] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-8] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-8] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-8] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-8] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-14] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-14] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-14] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-14] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-14] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-14] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-12] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-12] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-12] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-12] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-12] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-12] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-13] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-13] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-13] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-13] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-13] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-13] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-0] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-0] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-0] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-0] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-0] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-0] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-9] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-9] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-9] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-9] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-9] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-9] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-6] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-6] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-6] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-6] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-6] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-6] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-16] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-16] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-16] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-16] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-16] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-16] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-15] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-15] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-15] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-15] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-15] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-15] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-11] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-11] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-11] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-11] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-11] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-11] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-19] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-19] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-19] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-19] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-19] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-19] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-20] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-20] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-20] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-20] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-20] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-20] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-21] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-21] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-21] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-21] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-21] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-21] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-22] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-22] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-22] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-22] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-22] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-22] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-23] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-23] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-23] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-23] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-23] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-23] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-24] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-24] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-24] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-24] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-24] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-24] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-25] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-25] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-25] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-25] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-25] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-25] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-26] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-26] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-26] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-26] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-26] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-26] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-18] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-18] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-18] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-18] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:14 [jersey-client-async-executor-18] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:14 [jersey-client-async-executor-18] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:15 [jersey-client-async-executor-20] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:15 [jersey-client-async-executor-20] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:15 [jersey-client-async-executor-20] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:15 [jersey-client-async-executor-20] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:15 [jersey-client-async-executor-20] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:15 [jersey-client-async-executor-20] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:15 [jersey-client-async-executor-20] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:15 [jersey-client-async-executor-20] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:15 [jersey-client-async-executor-20] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:15 [jersey-client-async-executor-20] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:15 [jersey-client-async-executor-20] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:15 [jersey-client-async-executor-20] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
[INFO] Tests run: 7, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 4.194 s - in de.zalando.ep.zalenium.proxy.TestingBotRemoteProxyTest
[INFO] Running de.zalando.ep.zalenium.proxy.CloudTestingRemoteProxyTest
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.522 s - in de.zalando.ep.zalenium.proxy.CloudTestingRemoteProxyTest
[INFO] Running de.zalando.ep.zalenium.proxy.SauceLabsRemoteProxyTest
{server:CONFIG_UUID=null, browserName=safari, platformName=MAC}
[INFO] Tests run: 8, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 6.126 s - in de.zalando.ep.zalenium.proxy.SauceLabsRemoteProxyTest
[INFO] Running de.zalando.ep.zalenium.registry.ZaleniumRegistryTest
2020-03-19 19:44:24 [jersey-client-async-executor-0] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:24 [jersey-client-async-executor-0] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:24 [jersey-client-async-executor-0] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:24 [jersey-client-async-executor-0] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
2020-03-19 19:44:24 [jersey-client-async-executor-0] INFO  org.apache.http.impl.execchain.RetryExec  - I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:80: No such file or directory
2020-03-19 19:44:24 [jersey-client-async-executor-0] INFO  org.apache.http.impl.execchain.RetryExec  - Retrying request to {}->unix://localhost:80
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.634 s <<< FAILURE! - in de.zalando.ep.zalenium.registry.ZaleniumRegistryTest
[ERROR] addProxy(de.zalando.ep.zalenium.registry.ZaleniumRegistryTest)  Time elapsed: 0.634 s  <<< ERROR!
java.lang.NullPointerException
	at de.zalando.ep.zalenium.registry.ZaleniumRegistryTest.addProxy(ZaleniumRegistryTest.java:23)

[INFO] Running de.zalando.ep.zalenium.servlet.ZaleniumConsoleServletTest
<html>
    <head>
        <script src='/grid/resources/org/openqa/grid/images/jquery-3.1.1.min.js'></script>
        <script src='/grid/resources/org/openqa/grid/images/consoleservlet.js'></script>
        <link href='/grid/resources/org/openqa/grid/images/consoleservlet.css' rel='stylesheet' type='text/css' />
        <link href='/grid/resources/org/openqa/grid/images/favicon.ico' rel='icon' type='image/x-icon' />
        <meta http-equiv='refresh' content='1200' />
        <title>Grid Console</title>
        <style>
            .busy {opacity : 0.4; filter: alpha(opacity=40);}
        </style>
    </head>
    <body>
        <div id='main-content'>
            <div id='header'>
                <h1>
                    <a href='/grid/console'>Selenium</a>
                </h1>
                <h2>Grid Console v. 3.11.0</h2>
                <div>
                    <a id='helplink' target='_blank' href='https://github.com/SeleniumHQ/selenium/wiki/Grid2'>Help</a>
                </div>
            </div>
            <div id='left-column'>
                <div class='proxy'><p class='proxyname'>DockerSeleniumStarterRemoteProxy unknown version,Failed to connect to localhost/0:0:0:0:0:0:0:1:30000<p class='proxyid'>id : http://localhost:30000, OS : ANY</p><div class='tabs'><ul><li class='tab' type='browsers'><a title='test slots' href='#'>Browsers</a></li><li class='tab' type='config'><a title='node configuration' href='#'>Configuration</a></li></ul></div><div class='content'><div type='browsers' class='content_detail'><p class='protocol' >WebDriver</p><p>v:<a href='#'  title='{browserName=NEW_BROWSER, maxInstances=1, platformName=LINUX}'>NEW_BROWSER</a></p><p>v:<img src='/grid/resources/org/openqa/grid/images/firefox.png' width='16' height='16' title='{browserName=firefox, maxInstances=1, platformName=LINUX}' />
</p><p>v:<img src='/grid/resources/org/openqa/grid/images/chrome.png' width='16' height='16' title='{browserName=chrome, maxInstances=1, platformName=LINUX}' />
</p></div><div type='config' class='content_detail'><p>browserTimeout: 0</p><p>debug: false</p><p>port: 30000</p><p>role: node</p><p>timeout: 1800</p><p>cleanUpCycle: 5000</p><p>host: localhost</p><p>maxSession: 5</p><p>capabilities: Capabilities {browserName: firefox, maxInstances: 1, platformName: LINUX}</p><p>capabilities: Capabilities {browserName: chrome, maxInstances: 1, platformName: LINUX}</p><p>capabilities: Capabilities {browserName: NEW_BROWSER, maxInstances: 1, platformName: LINUX}</p><p>downPollingLimit: 2</p><p>hub: http://localhost:4444</p><p>nodePolling: 5000</p><p>nodeStatusCheckTimeout: 5000</p><p>proxy: de.zalando.ep.zalenium.proxy.DockerSeleniumStarterRemoteProxy</p><p>register: true</p><p>registerCycle: 5000</p><p>remoteHost: http://localhost:30000</p><p>unregisterIfStillDownAfter: 60000</p></div></div></div><div class='proxy'><p class='proxyname'>DockerSeleniumRemoteProxy unknown version,Failed to connect to localhost/0:0:0:0:0:0:0:1:40000<p class='proxyid'>id : http://localhost:40000, OS : ANY</p><div class='tabs'><ul><li class='tab' type='browsers'><a title='test slots' href='#'>Browsers</a></li><li class='tab' type='config'><a title='node configuration' href='#'>Configuration</a></li></ul></div><div class='content'><div type='browsers' class='content_detail'><p class='protocol' >WebDriver</p><p>v:<a href='#'  title='{browserName=NEW_BROWSER, maxInstances=1, platformName=LINUX}'>NEW_BROWSER</a></p><p>v:<img src='/grid/resources/org/openqa/grid/images/firefox.png' width='16' height='16' title='{browserName=firefox, maxInstances=1, platformName=LINUX}' />
</p><p>v:<img src='/grid/resources/org/openqa/grid/images/chrome.png' width='16' height='16' title='{browserName=chrome, maxInstances=1, platformName=LINUX}' />
</p></div><div type='config' class='content_detail'><p>browserTimeout: 0</p><p>debug: false</p><p>port: 40000</p><p>role: node</p><p>timeout: 1800</p><p>cleanUpCycle: 5000</p><p>host: localhost</p><p>maxSession: 5</p><p>capabilities: Capabilities {browserName: firefox, maxInstances: 1, platformName: LINUX}</p><p>capabilities: Capabilities {browserName: chrome, maxInstances: 1, platformName: LINUX}</p><p>capabilities: Capabilities {browserName: NEW_BROWSER, maxInstances: 1, platformName: LINUX}</p><p>downPollingLimit: 2</p><p>hub: http://localhost:4444</p><p>nodePolling: 5000</p><p>nodeStatusCheckTimeout: 5000</p><p>proxy: de.zalando.ep.zalenium.proxy.DockerSeleniumRemoteProxy</p><p>register: true</p><p>registerCycle: 5000</p><p>remoteHost: http://localhost:40000</p><p>unregisterIfStillDownAfter: 60000</p></div></div></div><div class='proxy'><p class='proxyname'>DockerSeleniumRemoteProxy unknown version,Failed to connect to localhost/0:0:0:0:0:0:0:1:40001<p class='proxyid'>id : http://localhost:40001, OS : ANY</p><div class='tabs'><ul><li class='tab' type='browsers'><a title='test slots' href='#'>Browsers</a></li><li class='tab' type='config'><a title='node configuration' href='#'>Configuration</a></li></ul></div><div class='content'><div type='browsers' class='content_detail'><p class='protocol' >WebDriver</p><p>v:<a href='#'  title='{browserName=NEW_BROWSER, maxInstances=1, platformName=LINUX}'>NEW_BROWSER</a><a href='#'  title='{browserName=NEW_BROWSER, maxInstances=1, platformName=LINUX}'>NEW_BROWSER</a></p><p>v:<img src='/grid/resources/org/openqa/grid/images/firefox.png' width='16' height='16' title='{browserName=firefox, maxInstances=1, platformName=LINUX}' />
</p><p>v:<img src='/grid/resources/org/openqa/grid/images/chrome.png' width='16' height='16' title='{browserName=chrome, maxInstances=1, platformName=LINUX}' />
</p></div><div type='config' class='content_detail'><p>browserTimeout: 0</p><p>debug: false</p><p>port: 40001</p><p>role: node</p><p>timeout: 1800</p><p>cleanUpCycle: 5000</p><p>host: localhost</p><p>maxSession: 5</p><p>capabilities: Capabilities {browserName: firefox, maxInstances: 1, platformName: LINUX}</p><p>capabilities: Capabilities {browserName: chrome, maxInstances: 1, platformName: LINUX}</p><p>capabilities: Capabilities {browserName: NEW_BROWSER, maxInstances: 1, platformName: LINUX}</p><p>capabilities: Capabilities {browserName: NEW_BROWSER, maxInstances: 1, platformName: LINUX}</p><p>downPollingLimit: 2</p><p>hub: http://localhost:4444</p><p>nodePolling: 5000</p><p>nodeStatusCheckTimeout: 5000</p><p>proxy: de.zalando.ep.zalenium.proxy.DockerSeleniumRemoteProxy</p><p>register: true</p><p>registerCycle: 5000</p><p>remoteHost: http://localhost:40001</p><p>unregisterIfStillDownAfter: 60000</p></div></div></div>
            </div>
            <div id='right-column'>
                <div class='proxy'>
    <p class='proxyname'>
    SauceLabsRemoteProxy  (version : Sauce Labs)
    <p class='proxyid'>
        id : http://localhost:30001, OS : ANY
    </p>
    <!-- Node Tabs -->
    <div class='tabs'>
        <ul>
            <li class='tab' type='browsers'>
                <a title='test slots' href='#'>Browsers</a>
            </li>
            <li class='tab' type='config'>
                <a title='node configuration' href='#'>Configuration</a>
            </li>
        </ul>
    </div>
    <div class='content'>
        <div type='browsers' class='content_detail'>
            <p class='protocol'>
                WebDriver
            </p>
            
            <p>
                v:
                
                <img src='/grid/admin/ZaleniumResourceServlet/images/saucelabs.png' width='16' height='16' class='' title='{browserName=SauceLabs, maxInstances=1, platformName=ANY}'/>
                
            </p>
            
        </div>
        <div type='config' class='content_detail'>
            <p><p>browserTimeout: 0</p><p>debug: false</p><p>port: 30001</p><p>role: node</p><p>timeout: 1800</p><p>cleanUpCycle: 5000</p><p>host: localhost</p><p>maxSession: 1</p><p>capabilities: Capabilities {browserName: SauceLabs, maxInstances: 1, platformName: ANY}</p><p>downPollingLimit: 2</p><p>hub: http://localhost:4444</p><p>nodePolling: 5000</p><p>nodeStatusCheckTimeout: 5000</p><p>proxy: de.zalando.ep.zalenium.proxy.SauceLabsRemoteProxy</p><p>register: true</p><p>registerCycle: 5000</p><p>remoteHost: http://localhost:30001</p><p>unregisterIfStillDownAfter: 60000</p></p>
        </div>
    </div>
</div>

<div class='proxy'>
    <p class='proxyname'>
    BrowserStackRemoteProxy  (version : 2.45.0)
    <p class='proxyid'>
        id : http://localhost:30002, OS : ANY
    </p>
    <!-- Node Tabs -->
    <div class='tabs'>
        <ul>
            <li class='tab' type='browsers'>
                <a title='test slots' href='#'>Browsers</a>
            </li>
            <li class='tab' type='config'>
                <a title='node configuration' href='#'>Configuration</a>
            </li>
        </ul>
    </div>
    <div class='content'>
        <div type='browsers' class='content_detail'>
            <p class='protocol'>
                WebDriver
            </p>
            
            <p>
                v:
                
                <img src='/grid/admin/ZaleniumResourceServlet/images/browserstack.png' width='16' height='16' class='' title='{browserName=BrowserStack, maxInstances=1, platformName=ANY}'/>
                
            </p>
            
        </div>
        <div type='config' class='content_detail'>
            <p><p>browserTimeout: 0</p><p>debug: false</p><p>port: 30002</p><p>role: node</p><p>timeout: 1800</p><p>cleanUpCycle: 5000</p><p>host: localhost</p><p>maxSession: 1</p><p>capabilities: Capabilities {browserName: BrowserStack, maxInstances: 1, platformName: ANY}</p><p>downPollingLimit: 2</p><p>hub: http://localhost:4444</p><p>nodePolling: 5000</p><p>nodeStatusCheckTimeout: 5000</p><p>proxy: de.zalando.ep.zalenium.proxy.BrowserStackRemoteProxy</p><p>register: true</p><p>registerCycle: 5000</p><p>remoteHost: http://localhost:30002</p><p>unregisterIfStillDownAfter: 60000</p></p>
        </div>
    </div>
</div>

<div class='proxy'>
    <p class='proxyname'>
    TestingBotRemoteProxy  (version : 3.14.0)
    <p class='proxyid'>
        id : http://localhost:30003, OS : ANY
    </p>
    <!-- Node Tabs -->
    <div class='tabs'>
        <ul>
            <li class='tab' type='browsers'>
                <a title='test slots' href='#'>Browsers</a>
            </li>
            <li class='tab' type='config'>
                <a title='node configuration' href='#'>Configuration</a>
            </li>
        </ul>
    </div>
    <div class='content'>
        <div type='browsers' class='content_detail'>
            <p class='protocol'>
                WebDriver
            </p>
            
            <p>
                v:
                
                <img src='/grid/admin/ZaleniumResourceServlet/images/testingbot.png' width='16' height='16' class='' title='{browserName=TestingBot, maxInstances=1, platformName=ANY}'/>
                
            </p>
            
        </div>
        <div type='config' class='content_detail'>
            <p><p>browserTimeout: 0</p><p>debug: false</p><p>port: 30003</p><p>role: node</p><p>timeout: 1800</p><p>cleanUpCycle: 5000</p><p>host: localhost</p><p>maxSession: 1</p><p>capabilities: Capabilities {browserName: TestingBot, maxInstances: 1, platformName: ANY}</p><p>downPollingLimit: 2</p><p>hub: http://localhost:4444</p><p>nodePolling: 5000</p><p>nodeStatusCheckTimeout: 5000</p><p>proxy: de.zalando.ep.zalenium.proxy.TestingBotRemoteProxy</p><p>register: true</p><p>registerCycle: 5000</p><p>remoteHost: http://localhost:30003</p><p>unregisterIfStillDownAfter: 60000</p></p>
        </div>
    </div>
</div>


            </div>
            <div class='clearfix'></div>
            <div>
                
                <ul>
                    
                </ul>
            </div>
            <a hidden href='?config=true&configDebug=true'>view config</a>
            <div  id='hub-config'>
                
                <b>Config for the hub :</b><br/>
                <abbr title='browserTimeout'>browserTimeout : </abbr>0</br><abbr title='debug'>debug : </abbr>false</br><abbr title='port'>port : </abbr>4444</br><abbr title='role'>role : </abbr>hub</br><abbr title='timeout'>timeout : </abbr>1800</br><abbr title='cleanUpCycle'>cleanUpCycle : </abbr>5000</br><abbr title='host'>host : </abbr>172.27.246.62</br><abbr title='capabilityMatcher'>capabilityMatcher : </abbr>org.openqa.grid.internal.utils.DefaultCapabilityMatcher</br><abbr title='newSessionWaitTimeout'>newSessionWaitTimeout : </abbr>-1</br><abbr title='throwOnCapabilityNotPresent'>throwOnCapabilityNotPresent : </abbr>true</br><abbr title='registry'>registry : </abbr>org.openqa.grid.internal.DefaultGridRegistry</br>
                <div  id='hub-config-verbose'>
                    <b>Config details :</b><br/>
                    <b>hub launched with :</b>
                    <abbr title='browserTimeout'>browserTimeout : </abbr>0</br><abbr title='debug'>debug : </abbr>false</br><abbr title='port'>port : </abbr>4444</br><abbr title='role'>role : </abbr>hub</br><abbr title='timeout'>timeout : </abbr>1800</br><abbr title='cleanUpCycle'>cleanUpCycle : </abbr>5000</br><abbr title='host'>host : </abbr>172.27.246.62</br><abbr title='capabilityMatcher'>capabilityMatcher : </abbr>org.openqa.grid.internal.utils.DefaultCapabilityMatcher</br><abbr title='newSessionWaitTimeout'>newSessionWaitTimeout : </abbr>-1</br><abbr title='throwOnCapabilityNotPresent'>throwOnCapabilityNotPresent : </abbr>true</br><abbr title='registry'>registry : </abbr>org.openqa.grid.internal.DefaultGridRegistry</br>
                    <br/><b>the final configuration comes from :</b><br/>
                    <b>the default :</b><br/>
                    <abbr title='browserTimeout'>browserTimeout : </abbr>0</br><abbr title='debug'>debug : </abbr>false</br><abbr title='port'>port : </abbr>4444</br><abbr title='role'>role : </abbr>hub</br><abbr title='timeout'>timeout : </abbr>1800</br><abbr title='cleanUpCycle'>cleanUpCycle : </abbr>5000</br><abbr title='capabilityMatcher'>capabilityMatcher : </abbr>org.openqa.grid.internal.utils.DefaultCapabilityMatcher</br><abbr title='newSessionWaitTimeout'>newSessionWaitTimeout : </abbr>-1</br><abbr title='throwOnCapabilityNotPresent'>throwOnCapabilityNotPresent : </abbr>true</br><abbr title='registry'>registry : </abbr>org.openqa.grid.internal.DefaultGridRegistry</br>
                    <br/><b>updated with params :</b><br/>
                    <abbr title='browserTimeout'>browserTimeout : </abbr>0</br><abbr title='debug'>debug : </abbr>false</br><abbr title='port'>port : </abbr>4444</br><abbr title='role'>role : </abbr>hub</br><abbr title='timeout'>timeout : </abbr>1800</br><abbr title='cleanUpCycle'>cleanUpCycle : </abbr>5000</br><abbr title='capabilityMatcher'>capabilityMatcher : </abbr>org.openqa.grid.internal.utils.DefaultCapabilityMatcher</br><abbr title='newSessionWaitTimeout'>newSessionWaitTimeout : </abbr>-1</br><abbr title='throwOnCapabilityNotPresent'>throwOnCapabilityNotPresent : </abbr>true</br><abbr title='registry'>registry : </abbr>org.openqa.grid.internal.DefaultGridRegistry</br>
                </div>
                
            </div>
        </div>
    </body>
</html>
[INFO] Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 4.358 s - in de.zalando.ep.zalenium.servlet.ZaleniumConsoleServletTest
[INFO] Running de.zalando.ep.zalenium.servlet.VncAuthenticationServletTest
[INFO] Tests run: 6, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 3.37 s - in de.zalando.ep.zalenium.servlet.VncAuthenticationServletTest
[INFO] Running de.zalando.ep.zalenium.servlet.LiveNodeServletTest
[INFO] Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.731 s - in de.zalando.ep.zalenium.servlet.LiveNodeServletTest
[INFO] Running de.zalando.ep.zalenium.dashboard.DashboardInformationServletTest
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.046 s - in de.zalando.ep.zalenium.dashboard.DashboardInformationServletTest
[INFO] Running de.zalando.ep.zalenium.dashboard.DashboardCleanupServletTest
[INFO] Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.033 s - in de.zalando.ep.zalenium.dashboard.DashboardCleanupServletTest
[INFO] Running de.zalando.ep.zalenium.dashboard.DashboardTest
[INFO] Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.029 s - in de.zalando.ep.zalenium.dashboard.DashboardTest
[INFO] 
[INFO] Results:
[INFO] 
[ERROR] Errors: 
[ERROR]   ZaleniumRegistryTest.addProxy:23 » NullPointer
[INFO] 
[ERROR] Tests run: 155, Failures: 0, Errors: 1, Skipped: 0
[INFO] 
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:50 min
[INFO] Finished at: 2020-03-19T19:44:34+01:00
[INFO] Final Memory: 18M/67M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.21.0:test (default-test) on project zalenium: There are test failures.
[ERROR] 
[ERROR] Please refer to /scratch/dginelli/workspace/repairnator-repairnator-experiments-zalando-zalenium-355877391-20180320-154108-firstCommit/target/surefire-reports for the individual test results.
[ERROR] Please refer to dump files (if any exist) [date]-jvmRun[N].dump, [date].dumpstream and [date]-jvmRun[N].dumpstream.
[ERROR] -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.21.0:test (default-test) on project zalenium: There are test failures.

Please refer to /scratch/dginelli/workspace/repairnator-repairnator-experiments-zalando-zalenium-355877391-20180320-154108-firstCommit/target/surefire-reports for the individual test results.
Please refer to dump files (if any exist) [date]-jvmRun[N].dump, [date].dumpstream and [date]-jvmRun[N].dumpstream.
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:212)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:153)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:145)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:116)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:80)
	at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:51)
	at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:307)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:193)
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:106)
	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:863)
	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:288)
	at org.apache.maven.cli.MavenCli.main(MavenCli.java:199)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)
Caused by: org.apache.maven.plugin.MojoFailureException: There are test failures.

Please refer to /scratch/dginelli/workspace/repairnator-repairnator-experiments-zalando-zalenium-355877391-20180320-154108-firstCommit/target/surefire-reports for the individual test results.
Please refer to dump files (if any exist) [date]-jvmRun[N].dump, [date].dumpstream and [date]-jvmRun[N].dumpstream.
	at org.apache.maven.plugin.surefire.SurefireHelper.throwException(SurefireHelper.java:240)
	at org.apache.maven.plugin.surefire.SurefireHelper.reportExecution(SurefireHelper.java:112)
	at org.apache.maven.plugin.surefire.SurefirePlugin.handleSummary(SurefirePlugin.java:354)
	at org.apache.maven.plugin.surefire.AbstractSurefireMojo.executeAfterPreconditionsChecked(AbstractSurefireMojo.java:1008)
	at org.apache.maven.plugin.surefire.AbstractSurefireMojo.execute(AbstractSurefireMojo.java:854)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:134)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:207)
	... 20 more
[ERROR] 
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
take a bit more until all get registered. 
2020-03-19 19:43:58 WARNING Cleaning up stale test sessions on the unregistered node http://localhost:30000 
2020-03-19 19:43:59 INFO   [DS] Starting new node for {browserName=firefox, platformName=ANY, screenResolution=1280x760}. 
2020-03-19 19:43:59 INFO   [DS] Container zalenium_40048 took longer than 60 seconds to be up and ready, this might be a signal that you have reached the hardware limits for the number of concurrent threads that you want to execute. 
2020-03-19 19:43:59 INFO   [DS] Container zalenium_40049 took longer than 60 seconds to be up and ready, this might be a signal that you have reached the hardware limits for the number of concurrent threads that you want to execute. 
2020-03-19 19:43:59 INFO   [DS] No container was created after 3 attempts, will wait until request is processed again... 
2020-03-19 19:44:00 WARNING Env. variable ZALENIUM_DESIRED_CONTAINERS is not a valid integer 
2020-03-19 19:44:00 WARNING Env. variable ZALENIUM_MAX_DOCKER_SELENIUM_CONTAINERS is not a valid integer 
2020-03-19 19:44:00 WARNING Env. variable ZALENIUM_SCREEN_WIDTH is not a valid integer 
2020-03-19 19:44:00 WARNING Env. variable ZALENIUM_SCREEN_HEIGHT is not a valid integer 
2020-03-19 19:44:00 WARNING ABC_NON_STANDARD_TIME_ZONE is not a real time zone. 
2020-03-19 19:44:00 INFO   Initialising Kubernetes support 
2020-03-19 19:44:00 INFO   MockWebServer[34241] starting to accept connections 
2020-03-19 19:44:00 INFO   MockWebServer[34241] received request: GET /api/v1/namespaces/test/pods/b-cn1002.hpc2n.umu.se HTTP/1.1 and responded: HTTP/1.1 200 OK 
2020-03-19 19:44:00 INFO   Kubernetes support initialised.
	Pod name: b-cn1002.hpc2n.umu.se
	app label: zalenium
	zalenium service name: zalenium
	Selenium Pod Resource Limits: {memory=Quantity(amount=4Gi, format=null, additionalProperties={}), cpu=Quantity(amount=500m, format=null, additionalProperties={})}
	Selenium Pod Resource Requests: {memory=Quantity(amount=1Gi, format=null, additionalProperties={}), cpu=Quantity(amount=250m, format=null, additionalProperties={})} 
2020-03-19 19:44:01 INFO   [DS] Starting new node for {browserName=firefox, platformName=LINUX}. 
2020-03-19 19:44:01 INFO   MockWebServer[34241] received request: GET /api/v1/namespaces/test/pods?labelSelector=createdBy%3Dzalenium HTTP/1.1 and responded: HTTP/1.1 200 OK 
2020-03-19 19:44:01 INFO   MockWebServer[34241] received request: GET /api/v1/namespaces/test/pods/zalenium_40026 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:01 INFO   MockWebServer[34241] received request: POST /api/v1/namespaces/test/pods HTTP/1.1 and responded: HTTP/1.1 201 OK 
2020-03-19 19:44:01 INFO   Initialising Kubernetes support 
2020-03-19 19:44:01 INFO   MockWebServer[56591] starting to accept connections 
2020-03-19 19:44:01 INFO   MockWebServer[34241] received request: GET /api/v1/namespaces/test/pods/zalenium-40050- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:01 INFO   MockWebServer[34241] received request: GET /api/v1/namespaces/test/pods/zalenium_40032 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:01 INFO   MockWebServer[34241] received request: GET /api/v1/namespaces/test/pods/zalenium_40003 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:01 INFO   MockWebServer[56591] received request: GET /api/v1/namespaces/test/pods/b-cn1002.hpc2n.umu.se HTTP/1.1 and responded: HTTP/1.1 200 OK 
2020-03-19 19:44:01 INFO   MockWebServer[34241] received request: GET /api/v1/namespaces/test/pods/zalenium_40005 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:01 INFO   MockWebServer[34241] received request: GET /api/v1/namespaces/test/pods/zalenium_40034 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:01 INFO   MockWebServer[34241] received request: GET /api/v1/namespaces/test/pods/zalenium_40007 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:01 INFO   Kubernetes support initialised.
	Pod name: b-cn1002.hpc2n.umu.se
	app label: zalenium
	zalenium service name: zalenium
	Selenium Pod Resource Limits: {memory=Quantity(amount=4Gi, format=null, additionalProperties={}), cpu=Quantity(amount=500m, format=null, additionalProperties={})}
	Selenium Pod Resource Requests: {memory=Quantity(amount=1Gi, format=null, additionalProperties={}), cpu=Quantity(amount=250m, format=null, additionalProperties={})} 
2020-03-19 19:44:01 INFO   MockWebServer[34241] received request: GET /api/v1/namespaces/test/pods/zalenium_40038 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:01 INFO   MockWebServer[34241] received request: GET /api/v1/namespaces/test/pods/zalenium_40040 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:01 INFO   MockWebServer[34241] received request: GET /api/v1/namespaces/test/pods/zalenium_40013 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:01 INFO   MockWebServer[34241] received request: GET /api/v1/namespaces/test/pods/zalenium-40000- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:01 INFO   MockWebServer[34241] received request: GET /api/v1/namespaces/test/pods/zalenium_40027 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:01 INFO   MockWebServer[34241] received request: GET /api/v1/namespaces/test/pods/zalenium_40028 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:01 INFO   MockWebServer[34241] received request: GET /api/v1/namespaces/test/pods/zalenium_40030 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:01 INFO   MockWebServer[34241] received request: GET /api/v1/namespaces/test/pods/zalenium_40036 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:01 INFO   Registered a node http://localhost:30000 
2020-03-19 19:44:01 INFO   About to clean up any left over selenium pods created by Zalenium 
2020-03-19 19:44:01 INFO   MockWebServer[56591] received request: GET /api/v1/namespaces/test/pods?labelSelector=createdBy%3Dzalenium HTTP/1.1 and responded: HTTP/1.1 200 OK 
2020-03-19 19:44:01 INFO   MockWebServer[56591] received request: DELETE /api/v1/namespaces/test/pods/b-cn1002.hpc2n.umu.se HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:01 INFO   [DS]  Setting up 0 nodes... 
2020-03-19 19:44:01 INFO   0 containers were created, it will take a bit more until all get registered. 
2020-03-19 19:44:01 WARNING Cleaning up stale test sessions on the unregistered node http://localhost:30000 
2020-03-19 19:44:01 INFO   Initialising Kubernetes support 
2020-03-19 19:44:01 INFO   MockWebServer[57691] starting to accept connections 
2020-03-19 19:44:01 INFO   MockWebServer[56591] received request: GET /api/v1/namespaces/test/pods/zalenium_40009 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:01 INFO   MockWebServer[56591] received request: GET /api/v1/namespaces/test/pods/zalenium_40011 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:01 INFO   MockWebServer[56591] received request: GET /api/v1/namespaces/test/pods/zalenium_40015 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:01 INFO   MockWebServer[56591] received request: GET /api/v1/namespaces/test/pods/zalenium_40017 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:01 INFO   MockWebServer[57691] received request: GET /api/v1/namespaces/test/pods/b-cn1002.hpc2n.umu.se HTTP/1.1 and responded: HTTP/1.1 200 OK 
2020-03-19 19:44:01 INFO   Kubernetes support initialised.
	Pod name: b-cn1002.hpc2n.umu.se
	app label: zalenium
	zalenium service name: zalenium
	Selenium Pod Resource Limits: {memory=Quantity(amount=4Gi, format=null, additionalProperties={}), cpu=Quantity(amount=500m, format=null, additionalProperties={})}
	Selenium Pod Resource Requests: {memory=Quantity(amount=1Gi, format=null, additionalProperties={}), cpu=Quantity(amount=250m, format=null, additionalProperties={})} 
2020-03-19 19:44:02 INFO   MockWebServer[56591] received request: GET /api/v1/namespaces/test/pods/zalenium_40026 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:02 INFO   MockWebServer[56591] received request: GET /api/v1/namespaces/test/pods/zalenium-40050- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:02 INFO   MockWebServer[56591] received request: GET /api/v1/namespaces/test/pods/zalenium_40032 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:02 INFO   MockWebServer[56591] received request: GET /api/v1/namespaces/test/pods/zalenium_40003 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:02 INFO   MockWebServer[56591] received request: GET /api/v1/namespaces/test/pods/zalenium_40005 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:02 INFO   MockWebServer[56591] received request: GET /api/v1/namespaces/test/pods/zalenium_40034 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:02 INFO   MockWebServer[56591] received request: GET /api/v1/namespaces/test/pods/zalenium_40007 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:02 INFO   MockWebServer[56591] received request: GET /api/v1/namespaces/test/pods/zalenium_40038 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:02 INFO   MockWebServer[56591] received request: GET /api/v1/namespaces/test/pods/zalenium_40040 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:02 INFO   MockWebServer[56591] received request: GET /api/v1/namespaces/test/pods/zalenium_40013 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:02 INFO   [DS] Starting new node for {tz=America/Montreal, browserName=firefox, platformName=ANY}. 
2020-03-19 19:44:02 INFO   MockWebServer[57691] received request: GET /api/v1/namespaces/test/pods?labelSelector=createdBy%3Dzalenium HTTP/1.1 and responded: HTTP/1.1 200 OK 
2020-03-19 19:44:02 INFO   MockWebServer[57691] received request: POST /api/v1/namespaces/test/pods HTTP/1.1 and responded: HTTP/1.1 201 OK 
2020-03-19 19:44:02 INFO   Initialising Kubernetes support 
2020-03-19 19:44:02 INFO   MockWebServer[49117] starting to accept connections 
2020-03-19 19:44:02 INFO   MockWebServer[57691] received request: GET /api/v1/namespaces/test/pods/zalenium-40051- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:02 INFO   MockWebServer[57691] received request: GET /api/v1/namespaces/test/pods/zalenium_40027 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:02 INFO   MockWebServer[57691] received request: GET /api/v1/namespaces/test/pods/zalenium_40028 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:02 INFO   MockWebServer[49117] received request: GET /api/v1/namespaces/test/pods/b-cn1002.hpc2n.umu.se HTTP/1.1 and responded: HTTP/1.1 200 OK 
2020-03-19 19:44:02 INFO   MockWebServer[57691] received request: GET /api/v1/namespaces/test/pods/zalenium_40030 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:02 INFO   Kubernetes support initialised.
	Pod name: b-cn1002.hpc2n.umu.se
	app label: zalenium
	zalenium service name: zalenium
	Selenium Pod Resource Limits: {memory=Quantity(amount=4Gi, format=null, additionalProperties={}), cpu=Quantity(amount=500m, format=null, additionalProperties={})}
	Selenium Pod Resource Requests: {memory=Quantity(amount=1Gi, format=null, additionalProperties={}), cpu=Quantity(amount=250m, format=null, additionalProperties={})} 
2020-03-19 19:44:02 INFO   MockWebServer[57691] received request: GET /api/v1/namespaces/test/pods/zalenium_40036 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:02 INFO   MockWebServer[57691] received request: GET /api/v1/namespaces/test/pods/zalenium_40009 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:02 INFO   MockWebServer[57691] received request: GET /api/v1/namespaces/test/pods/zalenium_40015 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:02 INFO   MockWebServer[57691] received request: GET /api/v1/namespaces/test/pods/zalenium_40017 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:02 INFO   MockWebServer[57691] received request: GET /api/v1/namespaces/test/pods/zalenium_40011 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:03 INFO   MockWebServer[57691] received request: GET /api/v1/namespaces/test/pods/zalenium_40026 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:03 INFO   MockWebServer[57691] received request: GET /api/v1/namespaces/test/pods/zalenium-40050- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:03 INFO   Initialising Kubernetes support 
2020-03-19 19:44:03 INFO   MockWebServer[48989] starting to accept connections 
2020-03-19 19:44:03 INFO   MockWebServer[49117] received request: GET /api/v1/namespaces/test/pods/zalenium_40005 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:03 INFO   MockWebServer[49117] received request: GET /api/v1/namespaces/test/pods/zalenium_40040 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:03 INFO   MockWebServer[49117] received request: GET /api/v1/namespaces/test/pods/zalenium_40013 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:03 INFO   MockWebServer[48989] received request: GET /api/v1/namespaces/test/pods/b-cn1002.hpc2n.umu.se HTTP/1.1 and responded: HTTP/1.1 200 OK 
2020-03-19 19:44:03 INFO   MockWebServer[49117] received request: GET /api/v1/namespaces/test/pods/zalenium_40032 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:03 INFO   MockWebServer[49117] received request: GET /api/v1/namespaces/test/pods/zalenium_40038 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:03 INFO   MockWebServer[49117] received request: GET /api/v1/namespaces/test/pods/zalenium_40034 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:03 INFO   MockWebServer[49117] received request: GET /api/v1/namespaces/test/pods/zalenium_40003 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:03 INFO   MockWebServer[49117] received request: GET /api/v1/namespaces/test/pods/zalenium_40007 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:03 INFO   Kubernetes support initialised.
	Pod name: b-cn1002.hpc2n.umu.se
	app label: zalenium
	zalenium service name: zalenium
	Selenium Pod Resource Limits: {memory=Quantity(amount=4Gi, format=null, additionalProperties={}), cpu=Quantity(amount=500m, format=null, additionalProperties={})}
	Selenium Pod Resource Requests: {memory=Quantity(amount=1Gi, format=null, additionalProperties={}), cpu=Quantity(amount=250m, format=null, additionalProperties={})} 
2020-03-19 19:44:03 INFO   MockWebServer[49117] received request: GET /api/v1/namespaces/test/pods/zalenium-40002- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:03 INFO   MockWebServer[49117] received request: GET /api/v1/namespaces/test/pods/zalenium-40001- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:03 INFO   MockWebServer[49117] received request: GET /api/v1/namespaces/test/pods/zalenium-40051- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:03 INFO   MockWebServer[49117] received request: GET /api/v1/namespaces/test/pods/zalenium_40027 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:03 INFO   MockWebServer[49117] received request: GET /api/v1/namespaces/test/pods/zalenium_40028 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:03 INFO   MockWebServer[49117] received request: GET /api/v1/namespaces/test/pods/zalenium_40030 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:03 INFO   MockWebServer[49117] received request: GET /api/v1/namespaces/test/pods/zalenium_40036 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:03 INFO   MockWebServer[49117] received request: GET /api/v1/namespaces/test/pods/zalenium_40009 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:03 INFO   MockWebServer[49117] received request: GET /api/v1/namespaces/test/pods/zalenium_40015 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:03 INFO   [DS] Starting new node for {screen-resolution=1500x1000, browserName=firefox, platformName=ANY}. 
2020-03-19 19:44:03 INFO   MockWebServer[48989] received request: GET /api/v1/namespaces/test/pods?labelSelector=createdBy%3Dzalenium HTTP/1.1 and responded: HTTP/1.1 200 OK 
2020-03-19 19:44:03 INFO   MockWebServer[48989] received request: POST /api/v1/namespaces/test/pods HTTP/1.1 and responded: HTTP/1.1 201 OK 
2020-03-19 19:44:03 INFO   Initialising Kubernetes support 
2020-03-19 19:44:03 INFO   MockWebServer[49177] starting to accept connections 
2020-03-19 19:44:03 INFO   MockWebServer[48989] received request: GET /api/v1/namespaces/test/pods/zalenium-40052- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:03 INFO   MockWebServer[48989] received request: GET /api/v1/namespaces/test/pods/zalenium_40017 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:04 INFO   MockWebServer[49177] received request: GET /api/v1/namespaces/test/pods/b-cn1002.hpc2n.umu.se HTTP/1.1 and responded: HTTP/1.1 200 OK 
2020-03-19 19:44:04 INFO   MockWebServer[48989] received request: GET /api/v1/namespaces/test/pods/zalenium_40011 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:04 INFO   MockWebServer[48989] received request: GET /api/v1/namespaces/test/pods/zalenium_40026 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:04 INFO   Kubernetes support initialised.
	Pod name: b-cn1002.hpc2n.umu.se
	app label: zalenium
	zalenium service name: zalenium
	Selenium Pod Resource Limits: {memory=Quantity(amount=4Gi, format=null, additionalProperties={}), cpu=Quantity(amount=500m, format=null, additionalProperties={})}
	Selenium Pod Resource Requests: {memory=Quantity(amount=1Gi, format=null, additionalProperties={}), cpu=Quantity(amount=250m, format=null, additionalProperties={})} 
2020-03-19 19:44:04 INFO   MockWebServer[48989] received request: GET /api/v1/namespaces/test/pods/zalenium-40050- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:04 INFO   MockWebServer[48989] received request: GET /api/v1/namespaces/test/pods/zalenium_40005 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:04 INFO   MockWebServer[48989] received request: GET /api/v1/namespaces/test/pods/zalenium_40040 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:04 INFO   MockWebServer[48989] received request: GET /api/v1/namespaces/test/pods/zalenium_40013 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:04 INFO   MockWebServer[48989] received request: GET /api/v1/namespaces/test/pods/zalenium_40032 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:04 INFO   MockWebServer[48989] received request: GET /api/v1/namespaces/test/pods/zalenium_40038 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:04 INFO   MockWebServer[48989] received request: GET /api/v1/namespaces/test/pods/zalenium_40034 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:04 INFO   MockWebServer[48989] received request: GET /api/v1/namespaces/test/pods/zalenium_40003 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:04 INFO   MockWebServer[48989] received request: GET /api/v1/namespaces/test/pods/zalenium_40007 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:04 INFO   MockWebServer[48989] received request: GET /api/v1/namespaces/test/pods/zalenium-40000- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:04 INFO   [DS] Request has waited 30 attempts for a node, something went wrong with the previous attempts, creating a new node for {browserName=firefox, platformName=LINUX}. 
2020-03-19 19:44:04 INFO   [DS] Starting new node for {browserName=firefox, platformName=LINUX}. 
2020-03-19 19:44:04 INFO   MockWebServer[49177] received request: GET /api/v1/namespaces/test/pods?labelSelector=createdBy%3Dzalenium HTTP/1.1 and responded: HTTP/1.1 200 OK 
2020-03-19 19:44:04 INFO   MockWebServer[49177] received request: GET /api/v1/namespaces/test/pods/zalenium-40051- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:04 INFO   Initialising Kubernetes support 
2020-03-19 19:44:04 INFO   MockWebServer[33225] starting to accept connections 
2020-03-19 19:44:04 INFO   MockWebServer[49177] received request: POST /api/v1/namespaces/test/pods HTTP/1.1 and responded: HTTP/1.1 201 OK 
2020-03-19 19:44:04 INFO   MockWebServer[49177] received request: GET /api/v1/namespaces/test/pods/zalenium-40053- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:04 INFO   MockWebServer[49177] received request: GET /api/v1/namespaces/test/pods/zalenium_40030 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:04 INFO   MockWebServer[49177] received request: GET /api/v1/namespaces/test/pods/zalenium_40027 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:04 INFO   MockWebServer[33225] received request: GET /api/v1/namespaces/test/pods/b-cn1002.hpc2n.umu.se HTTP/1.1 and responded: HTTP/1.1 200 OK 
2020-03-19 19:44:04 INFO   MockWebServer[49177] received request: GET /api/v1/namespaces/test/pods/zalenium_40028 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:04 INFO   Kubernetes support initialised.
	Pod name: b-cn1002.hpc2n.umu.se
	app label: zalenium
	zalenium service name: zalenium
	Selenium Pod Resource Limits: {memory=Quantity(amount=4Gi, format=null, additionalProperties={}), cpu=Quantity(amount=500m, format=null, additionalProperties={})}
	Selenium Pod Resource Requests: {memory=Quantity(amount=1Gi, format=null, additionalProperties={}), cpu=Quantity(amount=250m, format=null, additionalProperties={})} 
2020-03-19 19:44:04 INFO   MockWebServer[49177] received request: GET /api/v1/namespaces/test/pods/zalenium_40036 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:04 INFO   MockWebServer[49177] received request: GET /api/v1/namespaces/test/pods/zalenium_40009 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:04 INFO   MockWebServer[49177] received request: GET /api/v1/namespaces/test/pods/zalenium_40015 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:04 INFO   MockWebServer[49177] received request: GET /api/v1/namespaces/test/pods/zalenium-40052- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:04 INFO   MockWebServer[49177] received request: GET /api/v1/namespaces/test/pods/zalenium_40017 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:05 INFO   MockWebServer[49177] received request: GET /api/v1/namespaces/test/pods/zalenium_40011 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:05 INFO   MockWebServer[49177] received request: GET /api/v1/namespaces/test/pods/zalenium_40026 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:05 INFO   MockWebServer[49177] received request: GET /api/v1/namespaces/test/pods/zalenium-40050- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:05 INFO   Initialising Kubernetes support 
2020-03-19 19:44:05 INFO   MockWebServer[33389] starting to accept connections 
2020-03-19 19:44:05 INFO   MockWebServer[33225] received request: GET /api/v1/namespaces/test/pods/zalenium_40005 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:05 INFO   MockWebServer[33225] received request: GET /api/v1/namespaces/test/pods/zalenium_40040 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:05 INFO   MockWebServer[33225] received request: GET /api/v1/namespaces/test/pods/zalenium_40013 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:05 INFO   MockWebServer[33225] received request: GET /api/v1/namespaces/test/pods/zalenium_40032 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:05 INFO   MockWebServer[33225] received request: GET /api/v1/namespaces/test/pods/zalenium_40003 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:05 INFO   MockWebServer[33389] received request: GET /api/v1/namespaces/test/pods/b-cn1002.hpc2n.umu.se HTTP/1.1 and responded: HTTP/1.1 200 OK 
2020-03-19 19:44:05 INFO   MockWebServer[33225] received request: GET /api/v1/namespaces/test/pods/zalenium_40038 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:05 INFO   MockWebServer[33225] received request: GET /api/v1/namespaces/test/pods/zalenium_40034 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:05 INFO   Kubernetes support initialised.
	Pod name: b-cn1002.hpc2n.umu.se
	app label: zalenium
	zalenium service name: zalenium
	Selenium Pod Resource Limits: {memory=Quantity(amount=4Gi, format=null, additionalProperties={}), cpu=Quantity(amount=500m, format=null, additionalProperties={})}
	Selenium Pod Resource Requests: {memory=Quantity(amount=1Gi, format=null, additionalProperties={}), cpu=Quantity(amount=250m, format=null, additionalProperties={})} 
2020-03-19 19:44:05 INFO   MockWebServer[33225] received request: GET /api/v1/namespaces/test/pods/zalenium_40007 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:05 INFO   MockWebServer[33225] received request: GET /api/v1/namespaces/test/pods/zalenium-40051- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:05 INFO   MockWebServer[33225] received request: GET /api/v1/namespaces/test/pods/zalenium-40053- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:05 INFO   MockWebServer[33225] received request: GET /api/v1/namespaces/test/pods/zalenium_40030 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:05 INFO   MockWebServer[33225] received request: GET /api/v1/namespaces/test/pods/zalenium_40027 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:05 INFO   MockWebServer[33225] received request: GET /api/v1/namespaces/test/pods/zalenium_40028 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:05 INFO   MockWebServer[33225] received request: GET /api/v1/namespaces/test/pods/zalenium_40036 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:05 INFO   MockWebServer[33225] received request: GET /api/v1/namespaces/test/pods/zalenium_40009 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:05 INFO   MockWebServer[33225] received request: GET /api/v1/namespaces/test/pods/zalenium_40015 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:05 INFO   [DS] Starting new node for {browserName=chrome, platformName=ANY, resolution=-1300x800}. 
2020-03-19 19:44:05 INFO   MockWebServer[33389] received request: GET /api/v1/namespaces/test/pods?labelSelector=createdBy%3Dzalenium HTTP/1.1 and responded: HTTP/1.1 200 OK 
2020-03-19 19:44:05 INFO   MockWebServer[33389] received request: GET /api/v1/namespaces/test/pods/zalenium-40052- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:05 INFO   Initialising Kubernetes support 
2020-03-19 19:44:05 INFO   MockWebServer[33389] received request: POST /api/v1/namespaces/test/pods HTTP/1.1 and responded: HTTP/1.1 201 OK 
2020-03-19 19:44:05 INFO   MockWebServer[50505] starting to accept connections 
2020-03-19 19:44:05 INFO   MockWebServer[33389] received request: GET /api/v1/namespaces/test/pods/zalenium-40054- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:05 INFO   MockWebServer[33389] received request: GET /api/v1/namespaces/test/pods/zalenium_40017 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:06 INFO   MockWebServer[33389] received request: GET /api/v1/namespaces/test/pods/zalenium_40011 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:06 INFO   MockWebServer[50505] received request: GET /api/v1/namespaces/test/pods/b-cn1002.hpc2n.umu.se HTTP/1.1 and responded: HTTP/1.1 200 OK 
2020-03-19 19:44:06 INFO   MockWebServer[33389] received request: GET /api/v1/namespaces/test/pods/zalenium_40026 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:06 INFO   Kubernetes support initialised.
	Pod name: b-cn1002.hpc2n.umu.se
	app label: zalenium
	zalenium service name: zalenium
	Selenium Pod Resource Limits: {memory=Quantity(amount=4Gi, format=null, additionalProperties={}), cpu=Quantity(amount=500m, format=null, additionalProperties={})}
	Selenium Pod Resource Requests: {memory=Quantity(amount=1Gi, format=null, additionalProperties={}), cpu=Quantity(amount=250m, format=null, additionalProperties={})} 
2020-03-19 19:44:06 INFO   MockWebServer[33389] received request: GET /api/v1/namespaces/test/pods/zalenium-40050- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:06 INFO   MockWebServer[33389] received request: GET /api/v1/namespaces/test/pods/zalenium_40005 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:06 INFO   MockWebServer[33389] received request: GET /api/v1/namespaces/test/pods/zalenium_40040 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:06 INFO   MockWebServer[33389] received request: GET /api/v1/namespaces/test/pods/zalenium_40032 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:06 INFO   MockWebServer[33389] received request: GET /api/v1/namespaces/test/pods/zalenium_40003 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:06 INFO   MockWebServer[33389] received request: GET /api/v1/namespaces/test/pods/zalenium_40013 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:06 INFO   MockWebServer[33389] received request: GET /api/v1/namespaces/test/pods/zalenium_40038 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:06 INFO   MockWebServer[33389] received request: GET /api/v1/namespaces/test/pods/zalenium_40034 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:06 INFO   Unregistering the node http://localhost:30000 because it's been down for 60051 milliseconds 
2020-03-19 19:44:06 WARNING Cleaning up stale test sessions on the unregistered node http://localhost:30000 
2020-03-19 19:44:06 INFO   MockWebServer[33389] received request: GET /api/v1/namespaces/test/pods/zalenium-40002- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:06 INFO   MockWebServer[33389] received request: GET /api/v1/namespaces/test/pods/zalenium_40007 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:06 INFO   MockWebServer[33389] received request: GET /api/v1/namespaces/test/pods/zalenium-40001- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:06 INFO   MockWebServer[33389] received request: GET /api/v1/namespaces/test/pods/zalenium-40051- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:06 INFO   Initialising Kubernetes support 
2020-03-19 19:44:06 INFO   MockWebServer[41177] starting to accept connections 
2020-03-19 19:44:06 INFO   MockWebServer[50505] received request: GET /api/v1/namespaces/test/pods/zalenium-40053- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:06 INFO   MockWebServer[41177] received request: GET /api/v1/namespaces/test/pods/b-cn1002.hpc2n.umu.se HTTP/1.1 and responded: HTTP/1.1 200 OK 
2020-03-19 19:44:06 INFO   MockWebServer[50505] received request: GET /api/v1/namespaces/test/pods/zalenium_40036 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:06 INFO   MockWebServer[50505] received request: GET /api/v1/namespaces/test/pods/zalenium_40027 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:06 INFO   MockWebServer[50505] received request: GET /api/v1/namespaces/test/pods/zalenium_40030 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:06 INFO   Kubernetes support initialised.
	Pod name: b-cn1002.hpc2n.umu.se
	app label: zalenium
	zalenium service name: zalenium
	Selenium Pod Resource Limits: {memory=Quantity(amount=4Gi, format=null, additionalProperties={}), cpu=Quantity(amount=500m, format=null, additionalProperties={})}
	Selenium Pod Resource Requests: {memory=Quantity(amount=1Gi, format=null, additionalProperties={}), cpu=Quantity(amount=250m, format=null, additionalProperties={})} 
2020-03-19 19:44:06 INFO   MockWebServer[50505] received request: GET /api/v1/namespaces/test/pods/zalenium_40028 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:06 INFO   MockWebServer[50505] received request: GET /api/v1/namespaces/test/pods/zalenium_40009 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:06 INFO   MockWebServer[50505] received request: GET /api/v1/namespaces/test/pods/zalenium_40015 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:06 INFO   MockWebServer[50505] received request: GET /api/v1/namespaces/test/pods/zalenium-40052- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:06 INFO   MockWebServer[50505] received request: GET /api/v1/namespaces/test/pods/zalenium-40054- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:06 INFO   MockWebServer[50505] received request: GET /api/v1/namespaces/test/pods/zalenium_40017 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:07 INFO   MockWebServer[50505] received request: GET /api/v1/namespaces/test/pods/zalenium_40011 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:07 INFO   MockWebServer[50505] received request: GET /api/v1/namespaces/test/pods/zalenium_40026 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:07 INFO   MockWebServer[50505] received request: GET /api/v1/namespaces/test/pods/zalenium-40050- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:07 INFO   Initialising Kubernetes support 
2020-03-19 19:44:07 INFO   MockWebServer[48931] starting to accept connections 
2020-03-19 19:44:07 INFO   MockWebServer[41177] received request: GET /api/v1/namespaces/test/pods/zalenium_40005 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:07 INFO   MockWebServer[41177] received request: GET /api/v1/namespaces/test/pods/zalenium_40040 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:07 INFO   MockWebServer[41177] received request: GET /api/v1/namespaces/test/pods/zalenium_40032 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:07 INFO   MockWebServer[41177] received request: GET /api/v1/namespaces/test/pods/zalenium_40003 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:07 INFO   MockWebServer[41177] received request: GET /api/v1/namespaces/test/pods/zalenium_40013 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:07 INFO   MockWebServer[41177] received request: GET /api/v1/namespaces/test/pods/zalenium_40034 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:07 INFO   MockWebServer[48931] received request: GET /api/v1/namespaces/test/pods/b-cn1002.hpc2n.umu.se HTTP/1.1 and responded: HTTP/1.1 200 OK 
2020-03-19 19:44:07 INFO   MockWebServer[41177] received request: GET /api/v1/namespaces/test/pods/zalenium_40007 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:07 INFO   Kubernetes support initialised.
	Pod name: b-cn1002.hpc2n.umu.se
	app label: zalenium
	zalenium service name: zalenium
	Selenium Pod Resource Limits: {memory=Quantity(amount=4Gi, format=null, additionalProperties={}), cpu=Quantity(amount=500m, format=null, additionalProperties={})}
	Selenium Pod Resource Requests: {memory=Quantity(amount=1Gi, format=null, additionalProperties={}), cpu=Quantity(amount=250m, format=null, additionalProperties={})} 
2020-03-19 19:44:07 INFO   MockWebServer[41177] received request: GET /api/v1/namespaces/test/pods/zalenium_40038 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:07 INFO   MockWebServer[41177] received request: GET /api/v1/namespaces/test/pods/zalenium-40000- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:07 INFO   MockWebServer[41177] received request: GET /api/v1/namespaces/test/pods/zalenium-40051- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:07 INFO   Unregistering the node http://localhost:30000 because it's been down for 60032 milliseconds 
2020-03-19 19:44:07 WARNING Cleaning up stale test sessions on the unregistered node http://localhost:30000 
2020-03-19 19:44:07 INFO   MockWebServer[41177] received request: GET /api/v1/namespaces/test/pods/zalenium-40053- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:07 INFO   MockWebServer[41177] received request: GET /api/v1/namespaces/test/pods/zalenium_40036 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:07 INFO   MockWebServer[41177] received request: GET /api/v1/namespaces/test/pods/zalenium_40027 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:07 INFO   MockWebServer[41177] received request: GET /api/v1/namespaces/test/pods/zalenium_40030 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:07 INFO   MockWebServer[41177] received request: GET /api/v1/namespaces/test/pods/zalenium_40028 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:07 INFO   MockWebServer[41177] received request: GET /api/v1/namespaces/test/pods/zalenium_40009 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:07 INFO   MockWebServer[41177] received request: GET /api/v1/namespaces/test/pods/zalenium_40015 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:07 INFO   MockWebServer[41177] received request: GET /api/v1/namespaces/test/pods/zalenium-40052- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:07 INFO   [DS] Starting new node for {browserName=chrome, platformName=ANY, screenResolution=notAValidScreenResolution}. 
2020-03-19 19:44:07 INFO   MockWebServer[48931] received request: GET /api/v1/namespaces/test/pods?labelSelector=createdBy%3Dzalenium HTTP/1.1 and responded: HTTP/1.1 200 OK 
2020-03-19 19:44:07 INFO   Initialising Kubernetes support 
2020-03-19 19:44:07 INFO   MockWebServer[42925] starting to accept connections 
2020-03-19 19:44:07 INFO   MockWebServer[48931] received request: POST /api/v1/namespaces/test/pods HTTP/1.1 and responded: HTTP/1.1 201 OK 
2020-03-19 19:44:08 INFO   MockWebServer[48931] received request: GET /api/v1/namespaces/test/pods/zalenium-40055- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:08 INFO   MockWebServer[48931] received request: GET /api/v1/namespaces/test/pods/zalenium_40026 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:08 INFO   MockWebServer[48931] received request: GET /api/v1/namespaces/test/pods/zalenium_40017 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:08 INFO   MockWebServer[48931] received request: GET /api/v1/namespaces/test/pods/zalenium-40054- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:08 INFO   MockWebServer[42925] received request: GET /api/v1/namespaces/test/pods/b-cn1002.hpc2n.umu.se HTTP/1.1 and responded: HTTP/1.1 200 OK 
2020-03-19 19:44:08 INFO   MockWebServer[48931] received request: GET /api/v1/namespaces/test/pods/zalenium_40011 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:08 INFO   Kubernetes support initialised.
	Pod name: b-cn1002.hpc2n.umu.se
	app label: zalenium
	zalenium service name: zalenium
	Selenium Pod Resource Limits: {memory=Quantity(amount=4Gi, format=null, additionalProperties={}), cpu=Quantity(amount=500m, format=null, additionalProperties={})}
	Selenium Pod Resource Requests: {memory=Quantity(amount=1Gi, format=null, additionalProperties={}), cpu=Quantity(amount=250m, format=null, additionalProperties={})} 
2020-03-19 19:44:08 INFO   MockWebServer[48931] received request: GET /api/v1/namespaces/test/pods/zalenium-40050- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:08 INFO   MockWebServer[48931] received request: GET /api/v1/namespaces/test/pods/zalenium_40005 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:08 INFO   MockWebServer[48931] received request: GET /api/v1/namespaces/test/pods/zalenium_40040 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:08 INFO   MockWebServer[48931] received request: GET /api/v1/namespaces/test/pods/zalenium_40003 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:08 INFO   MockWebServer[48931] received request: GET /api/v1/namespaces/test/pods/zalenium_40032 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:08 INFO   MockWebServer[48931] received request: GET /api/v1/namespaces/test/pods/zalenium_40013 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:08 INFO   MockWebServer[48931] received request: GET /api/v1/namespaces/test/pods/zalenium_40034 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:08 INFO   MockWebServer[48931] received request: GET /api/v1/namespaces/test/pods/zalenium_40007 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:08 INFO   MockWebServer[48931] received request: GET /api/v1/namespaces/test/pods/zalenium_40038 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:08 INFO   MockWebServer[48931] received request: GET /api/v1/namespaces/test/pods/zalenium-40051- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:08 INFO   [DS] Starting new node for {browserName=chrome, platformName=ANY, resolution=1300x900}. 
2020-03-19 19:44:08 INFO   MockWebServer[42925] received request: GET /api/v1/namespaces/test/pods?labelSelector=createdBy%3Dzalenium HTTP/1.1 and responded: HTTP/1.1 200 OK 
2020-03-19 19:44:08 INFO   Initialising Kubernetes support 
2020-03-19 19:44:08 INFO   MockWebServer[42925] received request: POST /api/v1/namespaces/test/pods HTTP/1.1 and responded: HTTP/1.1 201 OK 
2020-03-19 19:44:08 INFO   MockWebServer[52183] starting to accept connections 
2020-03-19 19:44:08 INFO   MockWebServer[42925] received request: GET /api/v1/namespaces/test/pods/zalenium-40056- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:08 INFO   MockWebServer[42925] received request: GET /api/v1/namespaces/test/pods/zalenium-40053- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:08 INFO   MockWebServer[52183] received request: GET /api/v1/namespaces/test/pods/b-cn1002.hpc2n.umu.se HTTP/1.1 and responded: HTTP/1.1 200 OK 
2020-03-19 19:44:08 INFO   MockWebServer[42925] received request: GET /api/v1/namespaces/test/pods/zalenium_40036 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:08 INFO   MockWebServer[42925] received request: GET /api/v1/namespaces/test/pods/zalenium_40028 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:08 INFO   Kubernetes support initialised.
	Pod name: b-cn1002.hpc2n.umu.se
	app label: zalenium
	zalenium service name: zalenium
	Selenium Pod Resource Limits: {memory=Quantity(amount=4Gi, format=null, additionalProperties={}), cpu=Quantity(amount=500m, format=null, additionalProperties={})}
	Selenium Pod Resource Requests: {memory=Quantity(amount=1Gi, format=null, additionalProperties={}), cpu=Quantity(amount=250m, format=null, additionalProperties={})} 
2020-03-19 19:44:08 INFO   MockWebServer[42925] received request: GET /api/v1/namespaces/test/pods/zalenium_40009 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:08 INFO   MockWebServer[42925] received request: GET /api/v1/namespaces/test/pods/zalenium_40027 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:08 INFO   MockWebServer[42925] received request: GET /api/v1/namespaces/test/pods/zalenium_40030 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:08 INFO   MockWebServer[42925] received request: GET /api/v1/namespaces/test/pods/zalenium_40015 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:08 INFO   MockWebServer[42925] received request: GET /api/v1/namespaces/test/pods/zalenium-40052- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:09 INFO   Unregistering the node http://localhost:30000 because it's been down for 60482 milliseconds 
2020-03-19 19:44:09 WARNING Cleaning up stale test sessions on the unregistered node http://localhost:30000 
2020-03-19 19:44:09 INFO   MockWebServer[42925] received request: GET /api/v1/namespaces/test/pods/zalenium-40055- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:09 INFO   MockWebServer[42925] received request: GET /api/v1/namespaces/test/pods/zalenium_40026 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:09 INFO   MockWebServer[42925] received request: GET /api/v1/namespaces/test/pods/zalenium_40017 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:09 INFO   MockWebServer[42925] received request: GET /api/v1/namespaces/test/pods/zalenium-40054- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:09 INFO   Unregistering the node http://localhost:30000 because it's been down for 60031 milliseconds 
2020-03-19 19:44:09 WARNING Cleaning up stale test sessions on the unregistered node http://localhost:30000 
2020-03-19 19:44:09 INFO   MockWebServer[42925] received request: GET /api/v1/namespaces/test/pods/zalenium_40011 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:09 INFO   MockWebServer[42925] received request: GET /api/v1/namespaces/test/pods/zalenium-40050- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:09 INFO   Initialising Kubernetes support 
2020-03-19 19:44:09 INFO   MockWebServer[58827] starting to accept connections 
2020-03-19 19:44:09 INFO   MockWebServer[52183] received request: GET /api/v1/namespaces/test/pods/zalenium_40005 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:09 INFO   MockWebServer[52183] received request: GET /api/v1/namespaces/test/pods/zalenium_40040 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:09 INFO   MockWebServer[52183] received request: GET /api/v1/namespaces/test/pods/zalenium_40032 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:09 INFO   MockWebServer[52183] received request: GET /api/v1/namespaces/test/pods/zalenium_40003 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:09 INFO   MockWebServer[52183] received request: GET /api/v1/namespaces/test/pods/zalenium_40013 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:09 INFO   MockWebServer[52183] received request: GET /api/v1/namespaces/test/pods/zalenium_40034 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:09 INFO   MockWebServer[58827] received request: GET /api/v1/namespaces/test/pods/b-cn1002.hpc2n.umu.se HTTP/1.1 and responded: HTTP/1.1 200 OK 
2020-03-19 19:44:09 INFO   MockWebServer[52183] received request: GET /api/v1/namespaces/test/pods/zalenium-40002- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:09 INFO   MockWebServer[52183] received request: GET /api/v1/namespaces/test/pods/zalenium_40007 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:09 INFO   Kubernetes support initialised.
	Pod name: b-cn1002.hpc2n.umu.se
	app label: zalenium
	zalenium service name: zalenium
	Selenium Pod Resource Limits: {memory=Quantity(amount=4Gi, format=null, additionalProperties={}), cpu=Quantity(amount=500m, format=null, additionalProperties={})}
	Selenium Pod Resource Requests: {memory=Quantity(amount=1Gi, format=null, additionalProperties={}), cpu=Quantity(amount=250m, format=null, additionalProperties={})} 
2020-03-19 19:44:09 INFO   MockWebServer[52183] received request: GET /api/v1/namespaces/test/pods/zalenium_40038 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:09 INFO   MockWebServer[52183] received request: GET /api/v1/namespaces/test/pods/zalenium-40001- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:09 INFO   MockWebServer[52183] received request: GET /api/v1/namespaces/test/pods/zalenium-40051- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:09 INFO   Unregistering the node http://localhost:30000 because it's been down for 60047 milliseconds 
2020-03-19 19:44:09 WARNING Cleaning up stale test sessions on the unregistered node http://localhost:30000 
2020-03-19 19:44:09 INFO   MockWebServer[52183] received request: GET /api/v1/namespaces/test/pods/zalenium-40056- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:09 INFO   MockWebServer[52183] received request: GET /api/v1/namespaces/test/pods/zalenium-40053- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:09 INFO   MockWebServer[52183] received request: GET /api/v1/namespaces/test/pods/zalenium_40036 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:09 INFO   MockWebServer[52183] received request: GET /api/v1/namespaces/test/pods/zalenium_40028 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:09 INFO   MockWebServer[52183] received request: GET /api/v1/namespaces/test/pods/zalenium_40009 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:09 INFO   MockWebServer[52183] received request: GET /api/v1/namespaces/test/pods/zalenium_40027 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:09 INFO   MockWebServer[52183] received request: GET /api/v1/namespaces/test/pods/zalenium_40030 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:09 INFO   MockWebServer[52183] received request: GET /api/v1/namespaces/test/pods/zalenium_40015 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:09 INFO   MockWebServer[52183] received request: GET /api/v1/namespaces/test/pods/zalenium-40052- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:09 INFO   [DS] Starting new node for {browserName=chrome, platformName=LINUX}. 
2020-03-19 19:44:09 INFO   MockWebServer[58827] received request: GET /api/v1/namespaces/test/pods?labelSelector=createdBy%3Dzalenium HTTP/1.1 and responded: HTTP/1.1 200 OK 
2020-03-19 19:44:09 INFO   Initialising Kubernetes support 
2020-03-19 19:44:09 INFO   MockWebServer[58827] received request: POST /api/v1/namespaces/test/pods HTTP/1.1 and responded: HTTP/1.1 201 OK 
2020-03-19 19:44:09 INFO   MockWebServer[55137] starting to accept connections 
2020-03-19 19:44:10 INFO   MockWebServer[58827] received request: GET /api/v1/namespaces/test/pods/zalenium-40057- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:10 INFO   MockWebServer[58827] received request: GET /api/v1/namespaces/test/pods/zalenium-40055- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:10 INFO   MockWebServer[58827] received request: GET /api/v1/namespaces/test/pods/zalenium_40026 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:10 INFO   MockWebServer[58827] received request: GET /api/v1/namespaces/test/pods/zalenium_40017 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:10 INFO   MockWebServer[55137] received request: GET /api/v1/namespaces/test/pods/b-cn1002.hpc2n.umu.se HTTP/1.1 and responded: HTTP/1.1 200 OK 
2020-03-19 19:44:10 INFO   MockWebServer[58827] received request: GET /api/v1/namespaces/test/pods/zalenium-40054- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:10 INFO   MockWebServer[58827] received request: GET /api/v1/namespaces/test/pods/zalenium_40011 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:10 INFO   Kubernetes support initialised.
	Pod name: b-cn1002.hpc2n.umu.se
	app label: zalenium
	zalenium service name: zalenium
	Selenium Pod Resource Limits: {memory=Quantity(amount=4Gi, format=null, additionalProperties={}), cpu=Quantity(amount=500m, format=null, additionalProperties={})}
	Selenium Pod Resource Requests: {memory=Quantity(amount=1Gi, format=null, additionalProperties={}), cpu=Quantity(amount=250m, format=null, additionalProperties={})} 
2020-03-19 19:44:10 INFO   MockWebServer[58827] received request: GET /api/v1/namespaces/test/pods/zalenium-40050- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:10 INFO   Unregistering the node http://localhost:30000 because it's been down for 60046 milliseconds 
2020-03-19 19:44:10 WARNING Cleaning up stale test sessions on the unregistered node http://localhost:30000 
2020-03-19 19:44:10 INFO   MockWebServer[58827] received request: GET /api/v1/namespaces/test/pods/zalenium_40005 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:10 INFO   MockWebServer[58827] received request: GET /api/v1/namespaces/test/pods/zalenium_40040 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:10 INFO   MockWebServer[58827] received request: GET /api/v1/namespaces/test/pods/zalenium_40032 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:10 INFO   MockWebServer[58827] received request: GET /api/v1/namespaces/test/pods/zalenium_40013 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:10 INFO   MockWebServer[58827] received request: GET /api/v1/namespaces/test/pods/zalenium_40003 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:10 INFO   MockWebServer[58827] received request: GET /api/v1/namespaces/test/pods/zalenium_40007 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:10 INFO   MockWebServer[58827] received request: GET /api/v1/namespaces/test/pods/zalenium_40034 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:10 INFO   MockWebServer[58827] received request: GET /api/v1/namespaces/test/pods/zalenium_40038 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:10 INFO   MockWebServer[58827] received request: GET /api/v1/namespaces/test/pods/zalenium-40000- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:10 INFO   MockWebServer[58827] received request: GET /api/v1/namespaces/test/pods/zalenium-40051- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:10 WARNING null is not a real time zone. 
2020-03-19 19:44:10 INFO   Registered a node http://localhost:30000 
2020-03-19 19:44:10 INFO   About to clean up any left over selenium pods created by Zalenium 
2020-03-19 19:44:10 INFO   MockWebServer[55137] received request: GET /api/v1/namespaces/test/pods?labelSelector=createdBy%3Dzalenium HTTP/1.1 and responded: HTTP/1.1 200 OK 
2020-03-19 19:44:10 INFO   MockWebServer[55137] received request: DELETE /api/v1/namespaces/test/pods/b-cn1002.hpc2n.umu.se HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:10 INFO   [DS]  Setting up 7 nodes... 
2020-03-19 19:44:10 INFO   7 containers were created, it will take a bit more until all get registered. 
2020-03-19 19:44:10 WARNING Cleaning up stale test sessions on the unregistered node http://localhost:30000 
2020-03-19 19:44:10 INFO   MockWebServer[55137] received request: POST /api/v1/namespaces/test/pods HTTP/1.1 and responded: HTTP/1.1 201 OK 
2020-03-19 19:44:10 INFO   MockWebServer[55137] received request: POST /api/v1/namespaces/test/pods HTTP/1.1 and responded: HTTP/1.1 201 OK 
2020-03-19 19:44:10 INFO   MockWebServer[55137] received request: POST /api/v1/namespaces/test/pods HTTP/1.1 and responded: HTTP/1.1 201 OK 
2020-03-19 19:44:10 INFO   MockWebServer[55137] received request: POST /api/v1/namespaces/test/pods HTTP/1.1 and responded: HTTP/1.1 201 OK 
2020-03-19 19:44:10 INFO   MockWebServer[55137] received request: POST /api/v1/namespaces/test/pods HTTP/1.1 and responded: HTTP/1.1 201 OK 
2020-03-19 19:44:10 INFO   MockWebServer[55137] received request: GET /api/v1/namespaces/test/pods/zalenium-40053- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:10 INFO   MockWebServer[55137] received request: GET /api/v1/namespaces/test/pods/zalenium-40056- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:10 INFO   MockWebServer[55137] received request: POST /api/v1/namespaces/test/pods HTTP/1.1 and responded: HTTP/1.1 201 OK 
2020-03-19 19:44:10 INFO   Initialising Kubernetes support 
2020-03-19 19:44:10 INFO   MockWebServer[55137] received request: GET /api/v1/namespaces/test/pods/zalenium_40028 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:10 INFO   MockWebServer[55137] received request: POST /api/v1/namespaces/test/pods HTTP/1.1 and responded: HTTP/1.1 201 OK 
2020-03-19 19:44:10 INFO   MockWebServer[55137] received request: GET /api/v1/namespaces/test/pods/zalenium_40009 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:10 INFO   MockWebServer[55137] received request: GET /api/v1/namespaces/test/pods/zalenium_40027 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:10 INFO   MockWebServer[55137] received request: GET /api/v1/namespaces/test/pods/zalenium_40030 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:10 INFO   MockWebServer[38985] starting to accept connections 
2020-03-19 19:44:10 INFO   MockWebServer[55137] received request: GET /api/v1/namespaces/test/pods/zalenium_40036 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:10 INFO   MockWebServer[55137] received request: GET /api/v1/namespaces/test/pods/zalenium_40015 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:10 INFO   MockWebServer[55137] received request: GET /api/v1/namespaces/test/pods/zalenium-40052- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:10 INFO   MockWebServer[38985] received request: GET /api/v1/namespaces/test/pods/b-cn1002.hpc2n.umu.se HTTP/1.1 and responded: HTTP/1.1 200 OK 
2020-03-19 19:44:11 INFO   Kubernetes support initialised.
	Pod name: b-cn1002.hpc2n.umu.se
	app label: zalenium
	zalenium service name: zalenium
	Selenium Pod Resource Limits: {memory=Quantity(amount=4Gi, format=null, additionalProperties={}), cpu=Quantity(amount=500m, format=null, additionalProperties={})}
	Selenium Pod Resource Requests: {memory=Quantity(amount=1Gi, format=null, additionalProperties={}), cpu=Quantity(amount=250m, format=null, additionalProperties={})} 
2020-03-19 19:44:11 INFO   MockWebServer[55137] received request: GET /api/v1/namespaces/test/pods/zalenium-40057- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:11 INFO   MockWebServer[55137] received request: GET /api/v1/namespaces/test/pods/zalenium-40055- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:11 INFO   MockWebServer[55137] received request: GET /api/v1/namespaces/test/pods/zalenium_40026 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:11 INFO   MockWebServer[55137] received request: GET /api/v1/namespaces/test/pods/zalenium_40017 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:11 INFO   MockWebServer[55137] received request: GET /api/v1/namespaces/test/pods/zalenium-40054- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:11 INFO   MockWebServer[55137] received request: GET /api/v1/namespaces/test/pods/zalenium_40011 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:11 INFO   MockWebServer[55137] received request: GET /api/v1/namespaces/test/pods/zalenium-40050- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:11 INFO   MockWebServer[55137] received request: GET /api/v1/namespaces/test/pods/zalenium_40005 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:11 INFO   MockWebServer[55137] received request: GET /api/v1/namespaces/test/pods/zalenium_40040 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:11 INFO   MockWebServer[55137] received request: GET /api/v1/namespaces/test/pods/zalenium_40032 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:11 INFO   MockWebServer[55137] received request: GET /api/v1/namespaces/test/pods/zalenium_40013 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:11 INFO   MockWebServer[55137] received request: GET /api/v1/namespaces/test/pods/zalenium_40003 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:11 INFO   MockWebServer[55137] received request: GET /api/v1/namespaces/test/pods/zalenium_40007 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:11 INFO   MockWebServer[55137] received request: GET /api/v1/namespaces/test/pods/zalenium_40034 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:11 INFO   MockWebServer[55137] received request: GET /api/v1/namespaces/test/pods/zalenium_40038 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:11 INFO   [DS] Starting new node for {browserName=firefox, platformName=ANY, screenResolution=1280x760}. 
2020-03-19 19:44:11 INFO   MockWebServer[38985] received request: GET /api/v1/namespaces/test/pods?labelSelector=createdBy%3Dzalenium HTTP/1.1 and responded: HTTP/1.1 200 OK 
2020-03-19 19:44:11 INFO   MockWebServer[38985] received request: POST /api/v1/namespaces/test/pods HTTP/1.1 and responded: HTTP/1.1 201 OK 
2020-03-19 19:44:11 INFO   Initialising Kubernetes support 
2020-03-19 19:44:11 INFO   MockWebServer[48041] starting to accept connections 
2020-03-19 19:44:11 INFO   MockWebServer[38985] received request: GET /api/v1/namespaces/test/pods/zalenium-40065- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:11 INFO   MockWebServer[38985] received request: GET /api/v1/namespaces/test/pods/zalenium-40051- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:11 INFO   MockWebServer[48041] received request: GET /api/v1/namespaces/test/pods/b-cn1002.hpc2n.umu.se HTTP/1.1 and responded: HTTP/1.1 200 OK 
2020-03-19 19:44:11 INFO   Kubernetes support initialised.
	Pod name: b-cn1002.hpc2n.umu.se
	app label: zalenium
	zalenium service name: zalenium
	Selenium Pod Resource Limits: {memory=Quantity(amount=4Gi, format=null, additionalProperties={}), cpu=Quantity(amount=500m, format=null, additionalProperties={})}
	Selenium Pod Resource Requests: {memory=Quantity(amount=1Gi, format=null, additionalProperties={}), cpu=Quantity(amount=250m, format=null, additionalProperties={})} 
2020-03-19 19:44:11 INFO   MockWebServer[38985] received request: GET /api/v1/namespaces/test/pods/zalenium-40053- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:11 INFO   MockWebServer[38985] received request: GET /api/v1/namespaces/test/pods/zalenium-40056- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:11 INFO   MockWebServer[38985] received request: GET /api/v1/namespaces/test/pods/zalenium_40028 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:11 INFO   MockWebServer[38985] received request: GET /api/v1/namespaces/test/pods/zalenium_40009 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:11 INFO   MockWebServer[38985] received request: GET /api/v1/namespaces/test/pods/zalenium_40030 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:11 INFO   MockWebServer[38985] received request: GET /api/v1/namespaces/test/pods/zalenium_40027 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:11 INFO   MockWebServer[38985] received request: GET /api/v1/namespaces/test/pods/zalenium_40036 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:11 INFO   MockWebServer[38985] received request: GET /api/v1/namespaces/test/pods/zalenium_40015 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:11 INFO   MockWebServer[38985] received request: GET /api/v1/namespaces/test/pods/zalenium-40052- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:12 INFO   MockWebServer[38985] received request: GET /api/v1/namespaces/test/pods/zalenium-40057- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:12 INFO   MockWebServer[38985] received request: GET /api/v1/namespaces/test/pods/zalenium-40055- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:12 INFO   MockWebServer[38985] received request: GET /api/v1/namespaces/test/pods/zalenium_40026 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:12 INFO   MockWebServer[38985] received request: GET /api/v1/namespaces/test/pods/zalenium_40017 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:12 INFO   MockWebServer[38985] received request: GET /api/v1/namespaces/test/pods/zalenium-40054- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:12 INFO   MockWebServer[38985] received request: GET /api/v1/namespaces/test/pods/zalenium_40011 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:12 INFO   MockWebServer[38985] received request: GET /api/v1/namespaces/test/pods/zalenium-40050- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:12 INFO   Initialising Kubernetes support 
2020-03-19 19:44:12 INFO   MockWebServer[54583] starting to accept connections 
2020-03-19 19:44:12 INFO   MockWebServer[48041] received request: GET /api/v1/namespaces/test/pods/zalenium_40005 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:12 INFO   MockWebServer[48041] received request: GET /api/v1/namespaces/test/pods/zalenium_40040 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:12 INFO   MockWebServer[54583] received request: GET /api/v1/namespaces/test/pods/b-cn1002.hpc2n.umu.se HTTP/1.1 and responded: HTTP/1.1 200 OK 
2020-03-19 19:44:12 INFO   MockWebServer[48041] received request: GET /api/v1/namespaces/test/pods/zalenium_40032 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:12 INFO   Kubernetes support initialised.
	Pod name: b-cn1002.hpc2n.umu.se
	app label: zalenium
	zalenium service name: zalenium
	Selenium Pod Resource Limits: {memory=Quantity(amount=4Gi, format=null, additionalProperties={}), cpu=Quantity(amount=500m, format=null, additionalProperties={})}
	Selenium Pod Resource Requests: {memory=Quantity(amount=1Gi, format=null, additionalProperties={}), cpu=Quantity(amount=250m, format=null, additionalProperties={})} 
2020-03-19 19:44:12 INFO   MockWebServer[48041] received request: GET /api/v1/namespaces/test/pods/zalenium_40013 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:12 INFO   MockWebServer[48041] received request: GET /api/v1/namespaces/test/pods/zalenium-40002- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:12 INFO   MockWebServer[48041] received request: GET /api/v1/namespaces/test/pods/zalenium_40003 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:12 INFO   MockWebServer[48041] received request: GET /api/v1/namespaces/test/pods/zalenium_40007 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:12 INFO   MockWebServer[48041] received request: GET /api/v1/namespaces/test/pods/zalenium_40034 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:12 INFO   MockWebServer[48041] received request: GET /api/v1/namespaces/test/pods/zalenium_40038 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:12 INFO   MockWebServer[48041] received request: GET /api/v1/namespaces/test/pods/zalenium-40001- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:12 INFO   MockWebServer[48041] received request: GET /api/v1/namespaces/test/pods/zalenium-40065- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:12 INFO   MockWebServer[48041] received request: GET /api/v1/namespaces/test/pods/zalenium-40051- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:12 INFO   MockWebServer[48041] received request: GET /api/v1/namespaces/test/pods/zalenium-40053- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:12 INFO   MockWebServer[48041] received request: GET /api/v1/namespaces/test/pods/zalenium_40028 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:12 INFO   MockWebServer[48041] received request: GET /api/v1/namespaces/test/pods/zalenium_40027 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:12 INFO   MockWebServer[48041] received request: GET /api/v1/namespaces/test/pods/zalenium_40030 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:12 INFO   MockWebServer[48041] received request: GET /api/v1/namespaces/test/pods/zalenium_40036 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:12 INFO   MockWebServer[48041] received request: GET /api/v1/namespaces/test/pods/zalenium-40056- HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:12 INFO   MockWebServer[48041] received request: GET /api/v1/namespaces/test/pods/zalenium_40009 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:12 WARNING Env. variable ZALENIUM_DESIRED_CONTAINERS is not a valid integer 
2020-03-19 19:44:12 WARNING Env. variable ZALENIUM_MAX_DOCKER_SELENIUM_CONTAINERS is not a valid integer 
2020-03-19 19:44:12 WARNING Env. variable ZALENIUM_SCREEN_WIDTH is not a valid integer 
2020-03-19 19:44:12 WARNING Env. variable ZALENIUM_SCREEN_HEIGHT is not a valid integer 
2020-03-19 19:44:12 WARNING ABC_NON_STANDARD_TIME_ZONE is not a real time zone. 
2020-03-19 19:44:12 INFO   MockWebServer[48041] received request: GET /api/v1/namespaces/test/pods/zalenium_40015 HTTP/1.1 and responded: HTTP/1.1 404 Client Error 
2020-03-19 19:44:13 INFO   [TB] Account max. concurrency was NOT fetched from https://api.testingbot.com/v1/user 
2020-03-19 19:44:13 INFO   Registered a node http://hub.testingbot.com:80 
2020-03-19 19:44:13 INFO   Registered a node http://localhost:30000 
2020-03-19 19:44:13 INFO   [DS]  Setting up 0 nodes... 
2020-03-19 19:44:13 INFO   0 containers were created, it will take a bit more until all get registered. 
2020-03-19 19:44:13 INFO   Test will be forwarded to TestingBot, {browserName=chrome, platformName=MAC} 
2020-03-19 19:44:13 INFO   Currently using 0 paralell sessions towards TestingBot. Attempt to start one more. 
2020-03-19 19:44:13 INFO   Trying to create a new session on test slot {browserName=TestingBot, maxInstances=1, platformName=ANY} 
2020-03-19 19:44:14 SEVERE java.io.IOException: Server returned HTTP response code: 401 for URL: https://api.testingbot.com/v1/tests/2cf5d115-ca6f-4bc4-bc06-a4fca00836ce 
java.io.IOException: Server returned HTTP response code: 401 for URL: https://api.testingbot.com/v1/tests/2cf5d115-ca6f-4bc4-bc06-a4fca00836ce
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1894)
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1492)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:263)
	at de.zalando.ep.zalenium.util.CommonProxyUtilities.readJSONFromUrl(CommonProxyUtilities.java:43)
	at de.zalando.ep.zalenium.proxy.TestingBotRemoteProxy.getTestInformation(TestingBotRemoteProxy.java:93)
	at de.zalando.ep.zalenium.proxy.TestingBotRemoteProxy$MockitoMock$1293852747.getTestInformation$accessor$dF19ulJq(Unknown Source)
	at de.zalando.ep.zalenium.proxy.TestingBotRemoteProxy$MockitoMock$1293852747$auxiliary$2k2Ef5Mz.call(Unknown Source)
	at org.mockito.internal.invocation.RealMethod$FromCallable$1.call(RealMethod.java:40)
	at org.mockito.internal.invocation.RealMethod$FromBehavior.invoke(RealMethod.java:62)
	at org.mockito.internal.invocation.InterceptedInvocation.callRealMethod(InterceptedInvocation.java:127)
	at org.mockito.internal.stubbing.answers.CallsRealMethods.answer(CallsRealMethods.java:43)
	at org.mockito.Answers.answer(Answers.java:100)
	at org.mockito.internal.handler.MockHandlerImpl.handle(MockHandlerImpl.java:104)
	at org.mockito.internal.handler.NullResultGuardian.handle(NullResultGuardian.java:29)
	at org.mockito.internal.handler.InvocationNotifierHandler.handle(InvocationNotifierHandler.java:35)
	at org.mockito.internal.creation.bytebuddy.MockMethodInterceptor.doIntercept(MockMethodInterceptor.java:63)
	at org.mockito.internal.creation.bytebuddy.MockMethodInterceptor.doIntercept(MockMethodInterceptor.java:49)
	at org.mockito.internal.creation.bytebuddy.MockMethodInterceptor$DispatcherDefaultingToRealMethod.interceptSuperCallable(MockMethodInterceptor.java:110)
	at de.zalando.ep.zalenium.proxy.TestingBotRemoteProxy$MockitoMock$1293852747.getTestInformation(Unknown Source)
	at de.zalando.ep.zalenium.proxy.CloudTestingRemoteProxy.lambda$addTestToDashboard$3(CloudTestingRemoteProxy.java:237)
	at java.lang.Thread.run(Thread.java:748)

2020-03-19 19:44:14 INFO   Trying download once again from https://api.testingbot.com/v1/tests/2cf5d115-ca6f-4bc4-bc06-a4fca00836ce 
2020-03-19 19:44:14 INFO   [TB] Account max. concurrency was NOT fetched from https://api.testingbot.com/v1/user 
2020-03-19 19:44:14 INFO   Registered a node http://hub.testingbot.com:80 
2020-03-19 19:44:14 INFO   Registered a node http://localhost:30000 
2020-03-19 19:44:14 INFO   [DS]  Setting up 0 nodes... 
2020-03-19 19:44:14 INFO   0 containers were created, it will take a bit more until all get registered. 
2020-03-19 19:44:14 INFO   Test will be forwarded to TestingBot, {browserName=internet explorer, platformName=WIN8} 
2020-03-19 19:44:14 INFO   Currently using 0 paralell sessions towards TestingBot. Attempt to start one more. 
2020-03-19 19:44:14 INFO   Trying to create a new session on test slot {browserName=TestingBot, maxInstances=1, platformName=ANY} 
2020-03-19 19:44:14 INFO   [TB] Account max. concurrency was NOT fetched from https://api.testingbot.com/v1/user 
2020-03-19 19:44:14 INFO   Registered a node http://hub.testingbot.com:80 
2020-03-19 19:44:14 INFO   Registered a node http://localhost:30000 
2020-03-19 19:44:14 INFO   [DS]  Setting up 0 nodes... 
2020-03-19 19:44:14 INFO   0 containers were created, it will take a bit more until all get registered. 
2020-03-19 19:44:14 INFO   Test will be forwarded to TestingBot, {browserName=internet explorer, platformName=WIN8} 
2020-03-19 19:44:14 INFO   Currently using 0 paralell sessions towards TestingBot. Attempt to start one more. 
2020-03-19 19:44:14 INFO   Trying to create a new session on test slot {browserName=TestingBot, maxInstances=1, platformName=ANY} 
2020-03-19 19:44:14 INFO   Unregistering the node http://localhost:40000 because it's been down for 60038 milliseconds 
2020-03-19 19:44:15 INFO   [TB] Account max. concurrency was NOT fetched from https://api.testingbot.com/v1/user 
2020-03-19 19:44:15 INFO   Registered a node http://hub.testingbot.com:80 
2020-03-19 19:44:15 INFO   Registered a node http://localhost:30000 
2020-03-19 19:44:15 INFO   [DS]  Setting up 0 nodes... 
2020-03-19 19:44:15 INFO   0 containers were created, it will take a bit more until all get registered. 
2020-03-19 19:44:16 INFO   Unregistering the node http://localhost:40000 because it's been down for 60021 milliseconds 
2020-03-19 19:44:16 INFO   [TB] Account max. concurrency was NOT fetched from https://api.testingbot.com/v1/user 
2020-03-19 19:44:16 INFO   Registered a node http://hub.testingbot.com:80 
2020-03-19 19:44:16 INFO   Registered a node http://localhost:30000 
2020-03-19 19:44:16 INFO   [DS]  Setting up 0 nodes... 
2020-03-19 19:44:16 INFO   0 containers were created, it will take a bit more until all get registered. 
2020-03-19 19:44:16 INFO   Test will be forwarded to TestingBot, {browserName=internet explorer, platformName=WIN8} 
2020-03-19 19:44:16 INFO   Currently using 0 paralell sessions towards TestingBot. Attempt to start one more. 
2020-03-19 19:44:16 INFO   Trying to create a new session on test slot {browserName=TestingBot, maxInstances=1, platformName=ANY} 
2020-03-19 19:44:16 INFO   [TB] Account max. concurrency was NOT fetched from https://api.testingbot.com/v1/user 
2020-03-19 19:44:16 INFO   Registered a node http://hub.testingbot.com:80 
2020-03-19 19:44:16 INFO   Registered a node http://localhost:30000 
2020-03-19 19:44:16 INFO   [DS]  Setting up 0 nodes... 
2020-03-19 19:44:16 INFO   0 containers were created, it will take a bit more until all get registered. 
2020-03-19 19:44:17 INFO   [TB] Account max. concurrency was NOT fetched from https://api.testingbot.com/v1/user 
2020-03-19 19:44:17 INFO   Registered a node http://hub.testingbot.com:80 
2020-03-19 19:44:17 INFO   Registered a node http://localhost:30000 
2020-03-19 19:44:17 INFO   [DS]  Setting up 0 nodes... 
2020-03-19 19:44:17 INFO   0 containers were created, it will take a bit more until all get registered. 
2020-03-19 19:44:17 INFO   Test will be forwarded to TestingBot, {browserName=chrome, platformName=MAC} 
2020-03-19 19:44:17 INFO   Currently using 0 paralell sessions towards TestingBot. Attempt to start one more. 
2020-03-19 19:44:17 INFO   Trying to create a new session on test slot {browserName=TestingBot, maxInstances=1, platformName=ANY} 
2020-03-19 19:44:18 INFO   [SL] Account max. concurrency was NOT fetched from https://saucelabs.com/rest/v1/users/ 
2020-03-19 19:44:18 INFO   Registered a node http://ondemand.saucelabs.com:80 
2020-03-19 19:44:18 INFO   Registered a node http://localhost:30000 
2020-03-19 19:44:18 INFO   [DS]  Setting up 0 nodes... 
2020-03-19 19:44:18 INFO   0 containers were created, it will take a bit more until all get registered. 
2020-03-19 19:44:18 INFO   [SL] Account max. concurrency was NOT fetched from  
2020-03-19 19:44:18 INFO   [SL] Account max. concurrency was NOT fetched from https://saucelabs.com/rest/v1/users/ 
2020-03-19 19:44:18 INFO   Registered a node http://ondemand.saucelabs.com:80 
2020-03-19 19:44:18 INFO   Registered a node http://localhost:30000 
2020-03-19 19:44:18 INFO   [DS]  Setting up 0 nodes... 
2020-03-19 19:44:18 INFO   0 containers were created, it will take a bit more until all get registered. 
2020-03-19 19:44:18 INFO   Test will be forwarded to SauceLabs, {browserName=safari, platformName=MAC} 
2020-03-19 19:44:18 INFO   Currently using 0 paralell sessions towards SauceLabs. Attempt to start one more. 
2020-03-19 19:44:18 INFO   Trying to create a new session on test slot {browserName=SauceLabs, maxInstances=1, platformName=ANY} 
2020-03-19 19:44:19 SEVERE java.io.IOException: Server returned HTTP response code: 401 for URL: https://api.testingbot.com/v1/tests/2cf5d115-ca6f-4bc4-bc06-a4fca00836ce 
java.io.IOException: Server returned HTTP response code: 401 for URL: https://api.testingbot.com/v1/tests/2cf5d115-ca6f-4bc4-bc06-a4fca00836ce
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1894)
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1492)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:263)
	at de.zalando.ep.zalenium.util.CommonProxyUtilities.readJSONFromUrl(CommonProxyUtilities.java:43)
	at de.zalando.ep.zalenium.proxy.TestingBotRemoteProxy.getTestInformation(TestingBotRemoteProxy.java:93)
	at de.zalando.ep.zalenium.proxy.TestingBotRemoteProxy$MockitoMock$1293852747.getTestInformation$accessor$dF19ulJq(Unknown Source)
	at de.zalando.ep.zalenium.proxy.TestingBotRemoteProxy$MockitoMock$1293852747$auxiliary$2k2Ef5Mz.call(Unknown Source)
	at org.mockito.internal.invocation.RealMethod$FromCallable$1.call(RealMethod.java:40)
	at org.mockito.internal.invocation.RealMethod$FromBehavior.invoke(RealMethod.java:62)
	at org.mockito.internal.invocation.InterceptedInvocation.callRealMethod(InterceptedInvocation.java:127)
	at org.mockito.internal.stubbing.answers.CallsRealMethods.answer(CallsRealMethods.java:43)
	at org.mockito.Answers.answer(Answers.java:100)
	at org.mockito.internal.handler.MockHandlerImpl.handle(MockHandlerImpl.java:104)
	at org.mockito.internal.handler.NullResultGuardian.handle(NullResultGuardian.java:29)
	at org.mockito.internal.handler.InvocationNotifierHandler.handle(InvocationNotifierHandler.java:35)
	at org.mockito.internal.creation.bytebuddy.MockMethodInterceptor.doIntercept(MockMethodInterceptor.java:63)
	at org.mockito.internal.creation.bytebuddy.MockMethodInterceptor.doIntercept(MockMethodInterceptor.java:49)
	at org.mockito.internal.creation.bytebuddy.MockMethodInterceptor$DispatcherDefaultingToRealMethod.interceptSuperCallable(MockMethodInterceptor.java:110)
	at de.zalando.ep.zalenium.proxy.TestingBotRemoteProxy$MockitoMock$1293852747.getTestInformation(Unknown Source)
	at de.zalando.ep.zalenium.proxy.CloudTestingRemoteProxy.lambda$addTestToDashboard$3(CloudTestingRemoteProxy.java:237)
	at java.lang.Thread.run(Thread.java:748)

2020-03-19 19:44:19 INFO   Trying download once again from https://api.testingbot.com/v1/tests/2cf5d115-ca6f-4bc4-bc06-a4fca00836ce 
2020-03-19 19:44:19 INFO   [SL] Account max. concurrency was NOT fetched from https://saucelabs.com/rest/v1/users/ 
2020-03-19 19:44:19 INFO   Registered a node http://ondemand.saucelabs.com:80 
2020-03-19 19:44:19 INFO   Registered a node http://localhost:30000 
2020-03-19 19:44:19 INFO   [DS]  Setting up 0 nodes... 
2020-03-19 19:44:19 INFO   0 containers were created, it will take a bit more until all get registered. 
2020-03-19 19:44:19 INFO   Test will be forwarded to SauceLabs, {browserName=MicrosoftEdge, platformName=WIN10} 
2020-03-19 19:44:19 INFO   Currently using 0 paralell sessions towards SauceLabs. Attempt to start one more. 
2020-03-19 19:44:19 INFO   Trying to create a new session on test slot {browserName=SauceLabs, maxInstances=1, platformName=ANY} 
2020-03-19 19:44:19 SEVERE java.io.FileNotFoundException: https://saucelabs.com/rest/v1//jobs/testKey 
java.io.FileNotFoundException: https://saucelabs.com/rest/v1//jobs/testKey
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1890)
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1492)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:263)
	at de.zalando.ep.zalenium.util.CommonProxyUtilities.readJSONFromUrl(CommonProxyUtilities.java:43)
	at de.zalando.ep.zalenium.proxy.SauceLabsRemoteProxy.getTestInformation(SauceLabsRemoteProxy.java:107)
	at de.zalando.ep.zalenium.proxy.CloudTestingRemoteProxy.lambda$addTestToDashboard$3(CloudTestingRemoteProxy.java:237)
	at java.lang.Thread.run(Thread.java:748)

2020-03-19 19:44:19 INFO   Trying download once again from https://saucelabs.com/rest/v1//jobs/testKey 
2020-03-19 19:44:19 INFO   [SL] Account max. concurrency was NOT fetched from https://saucelabs.com/rest/v1/users/ 
2020-03-19 19:44:19 INFO   Registered a node http://ondemand.saucelabs.com:80 
2020-03-19 19:44:19 INFO   Registered a node http://localhost:30000 
2020-03-19 19:44:19 INFO   [DS]  Setting up 0 nodes... 
2020-03-19 19:44:19 INFO   0 containers were created, it will take a bit more until all get registered. 
2020-03-19 19:44:19 INFO   Unregistering the node http://localhost:40000 because it's been down for 60038 milliseconds 
2020-03-19 19:44:19 INFO   Unregistering the node http://localhost:40000 because it's been down for 65039 milliseconds 
2020-03-19 19:44:20 INFO   [SL] Account max. concurrency was NOT fetched from https://saucelabs.com/rest/v1/users/ 
2020-03-19 19:44:20 INFO   Registered a node http://ondemand.saucelabs.com:80 
2020-03-19 19:44:20 INFO   Registered a node http://localhost:30000 
2020-03-19 19:44:20 INFO   [DS]  Setting up 0 nodes... 
2020-03-19 19:44:20 INFO   0 containers were created, it will take a bit more until all get registered. 
2020-03-19 19:44:20 INFO   Test will be forwarded to SauceLabs, {browserName=safari, platformName=MAC} 
2020-03-19 19:44:20 INFO   Currently using 0 paralell sessions towards SauceLabs. Attempt to start one more. 
2020-03-19 19:44:20 INFO   Trying to create a new session on test slot {browserName=SauceLabs, maxInstances=1, platformName=ANY} 
2020-03-19 19:44:20 INFO   Unregistering the node http://localhost:40000 because it's been down for 60049 milliseconds 
2020-03-19 19:44:21 INFO   Unregistering the node http://localhost:40000 because it's been down for 60021 milliseconds 
2020-03-19 19:44:21 INFO   Unregistering the node http://localhost:40000 because it's been down for 65023 milliseconds 
2020-03-19 19:44:21 INFO   [SL] Account max. concurrency was NOT fetched from https://saucelabs.com/rest/v1/users/ 
2020-03-19 19:44:21 INFO   Registered a node http://ondemand.saucelabs.com:80 
2020-03-19 19:44:21 INFO   Registered a node http://localhost:30000 
2020-03-19 19:44:21 INFO   [DS]  Setting up 0 nodes... 
2020-03-19 19:44:21 INFO   0 containers were created, it will take a bit more until all get registered. 
2020-03-19 19:44:22 INFO   [SL] Account max. concurrency was NOT fetched from https://saucelabs.com/rest/v1/users/ 
2020-03-19 19:44:22 INFO   Registered a node http://ondemand.saucelabs.com:80 
2020-03-19 19:44:22 INFO   Registered a node http://localhost:30000 
2020-03-19 19:44:22 INFO   [DS]  Setting up 0 nodes... 
2020-03-19 19:44:22 INFO   0 containers were created, it will take a bit more until all get registered. 
2020-03-19 19:44:22 INFO   Test will be forwarded to SauceLabs, {browserName=chrome, platformName=MAC} 
2020-03-19 19:44:22 INFO   Currently using 0 paralell sessions towards SauceLabs. Attempt to start one more. 
2020-03-19 19:44:22 INFO   Trying to create a new session on test slot {browserName=SauceLabs, maxInstances=1, platformName=ANY} 
ffmpeg version 2.8.15-0ubuntu0.16.04.1 Copyright (c) 2000-2018 the FFmpeg developers
  built with gcc 5.4.0 (Ubuntu 5.4.0-6ubuntu1~16.04.10) 20160609
  configuration: --prefix=/usr --extra-version=0ubuntu0.16.04.1 --build-suffix=-ffmpeg --toolchain=hardened --libdir=/usr/lib/x86_64-linux-gnu --incdir=/usr/include/x86_64-linux-gnu --cc=cc --cxx=g++ --enable-gpl --enable-shared --disable-stripping --disable-decoder=libopenjpeg --disable-decoder=libschroedinger --enable-avresample --enable-avisynth --enable-gnutls --enable-ladspa --enable-libass --enable-libbluray --enable-libbs2b --enable-libcaca --enable-libcdio --enable-libflite --enable-libfontconfig --enable-libfreetype --enable-libfribidi --enable-libgme --enable-libgsm --enable-libmodplug --enable-libmp3lame --enable-libopenjpeg --enable-libopus --enable-libpulse --enable-librtmp --enable-libschroedinger --enable-libshine --enable-libsnappy --enable-libsoxr --enable-libspeex --enable-libssh --enable-libtheora --enable-libtwolame --enable-libvorbis --enable-libvpx --enable-libwavpack --enable-libwebp --enable-libx265 --enable-libxvid --enable-libzvbi --enable-openal --enable-opengl --enable-x11grab --enable-libdc1394 --enable-libiec61883 --enable-libzmq --enable-frei0r --enable-libx264 --enable-libopencv
  libavutil      54. 31.100 / 54. 31.100
  libavcodec     56. 60.100 / 56. 60.100
  libavformat    56. 40.101 / 56. 40.101
  libavdevice    56.  4.100 / 56.  4.100
  libavfilter     5. 40.101 /  5. 40.101
  libavresample   2.  1.  0 /  2.  1.  0
  libswscale      3.  1.101 /  3.  1.101
  libswresample   1.  2.101 /  1.  2.101
  libpostproc    53.  3.100 / 53.  3.100
/scratch/dginelli/workspace/repairnator-repairnator-experiments-zalando-zalenium-355877391-20180320-154108-firstCommit/target/videos/saucelabs_72e4f8ecf04440fe965faf657864ed52_googlechrome_Windows_2008_20200319194422248.flv: No such file or directory
2020-03-19 19:44:22 SEVERE org.apache.commons.exec.ExecuteException: Process exited with an error: 1 (Exit value: 1) 
org.apache.commons.exec.ExecuteException: Process exited with an error: 1 (Exit value: 1)
	at org.apache.commons.exec.DefaultExecutor.executeInternal(DefaultExecutor.java:404)
	at org.apache.commons.exec.DefaultExecutor.execute(DefaultExecutor.java:166)
	at org.apache.commons.exec.DefaultExecutor.execute(DefaultExecutor.java:153)
	at de.zalando.ep.zalenium.util.CommonProxyUtilities.convertFlvFileToMP4(CommonProxyUtilities.java:154)
	at de.zalando.ep.zalenium.util.CommonProxyUtilities$MockitoMock$1605122368.convertFlvFileToMP4$accessor$2IQK9V40(Unknown Source)
	at de.zalando.ep.zalenium.util.CommonProxyUtilities$MockitoMock$1605122368$auxiliary$FzAucOQF.call(Unknown Source)
	at org.mockito.internal.invocation.RealMethod$FromCallable$1.call(RealMethod.java:40)
	at org.mockito.internal.invocation.RealMethod$FromBehavior.invoke(RealMethod.java:62)
	at org.mockito.internal.invocation.InterceptedInvocation.callRealMethod(InterceptedInvocation.java:127)
	at org.mockito.internal.stubbing.answers.CallsRealMethods.answer(CallsRealMethods.java:43)
	at org.mockito.internal.stubbing.StubbedInvocationMatcher.answer(StubbedInvocationMatcher.java:36)
	at org.mockito.internal.handler.MockHandlerImpl.handle(MockHandlerImpl.java:97)
	at org.mockito.internal.handler.NullResultGuardian.handle(NullResultGuardian.java:29)
	at org.mockito.internal.handler.InvocationNotifierHandler.handle(InvocationNotifierHandler.java:35)
	at org.mockito.internal.creation.bytebuddy.MockMethodInterceptor.doIntercept(MockMethodInterceptor.java:63)
	at org.mockito.internal.creation.bytebuddy.MockMethodInterceptor.doIntercept(MockMethodInterceptor.java:49)
	at org.mockito.internal.creation.bytebuddy.MockMethodInterceptor$DispatcherDefaultingToRealMethod.interceptSuperCallable(MockMethodInterceptor.java:110)
	at de.zalando.ep.zalenium.util.CommonProxyUtilities$MockitoMock$1605122368.convertFlvFileToMP4(Unknown Source)
	at de.zalando.ep.zalenium.proxy.CloudTestingRemoteProxy.lambda$addTestToDashboard$3(CloudTestingRemoteProxy.java:245)
	at java.lang.Thread.run(Thread.java:748)

2020-03-19 19:44:22 INFO   [SL] Account max. concurrency was NOT fetched from https://saucelabs.com/rest/v1/users/ 
2020-03-19 19:44:22 INFO   Registered a node http://ondemand.saucelabs.com:80 
2020-03-19 19:44:22 INFO   Registered a node http://localhost:30000 
2020-03-19 19:44:22 INFO   [DS]  Setting up 0 nodes... 
2020-03-19 19:44:22 INFO   0 containers were created, it will take a bit more until all get registered. 
2020-03-19 19:44:22 INFO   Test will be forwarded to SauceLabs, {browserName=safari, platformName=MAC} 
2020-03-19 19:44:22 INFO   Currently using 0 paralell sessions towards SauceLabs. Attempt to start one more. 
2020-03-19 19:44:22 INFO   Trying to create a new session on test slot {browserName=SauceLabs, maxInstances=1, platformName=ANY} 
2020-03-19 19:44:23 INFO   Marking the node http://localhost:30000 as down: cannot reach the node for 2 tries 
2020-03-19 19:44:23 WARNING Tried to release session with internal key 609af2ac-0d9c-43ba-b95a-cb6e63d3a0de but couldn't find it. 
2020-03-19 19:44:23 INFO   SauceLabs Releasing slot and terminating session due to inactivity. 
2020-03-19 19:44:24 INFO   Marking the node http://localhost:30000 as down: cannot reach the node for 2 tries 
2020-03-19 19:44:24 SEVERE java.io.IOException: Server returned HTTP response code: 401 for URL: https://api.testingbot.com/v1/tests/2cf5d115-ca6f-4bc4-bc06-a4fca00836ce 
java.io.IOException: Server returned HTTP response code: 401 for URL: https://api.testingbot.com/v1/tests/2cf5d115-ca6f-4bc4-bc06-a4fca00836ce
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1894)
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1492)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:263)
	at de.zalando.ep.zalenium.util.CommonProxyUtilities.readJSONFromUrl(CommonProxyUtilities.java:43)
	at de.zalando.ep.zalenium.proxy.TestingBotRemoteProxy.getTestInformation(TestingBotRemoteProxy.java:93)
	at de.zalando.ep.zalenium.proxy.TestingBotRemoteProxy$MockitoMock$1293852747.getTestInformation$accessor$dF19ulJq(Unknown Source)
	at de.zalando.ep.zalenium.proxy.TestingBotRemoteProxy$MockitoMock$1293852747$auxiliary$2k2Ef5Mz.call(Unknown Source)
	at org.mockito.internal.invocation.RealMethod$FromCallable$1.call(RealMethod.java:40)
	at org.mockito.internal.invocation.RealMethod$FromBehavior.invoke(RealMethod.java:62)
	at org.mockito.internal.invocation.InterceptedInvocation.callRealMethod(InterceptedInvocation.java:127)
	at org.mockito.internal.stubbing.answers.CallsRealMethods.answer(CallsRealMethods.java:43)
	at org.mockito.Answers.answer(Answers.java:100)
	at org.mockito.internal.handler.MockHandlerImpl.handle(MockHandlerImpl.java:104)
	at org.mockito.internal.handler.NullResultGuardian.handle(NullResultGuardian.java:29)
	at org.mockito.internal.handler.InvocationNotifierHandler.handle(InvocationNotifierHandler.java:35)
	at org.mockito.internal.creation.bytebuddy.MockMethodInterceptor.doIntercept(MockMethodInterceptor.java:63)
	at org.mockito.internal.creation.bytebuddy.MockMethodInterceptor.doIntercept(MockMethodInterceptor.java:49)
	at org.mockito.internal.creation.bytebuddy.MockMethodInterceptor$DispatcherDefaultingToRealMethod.interceptSuperCallable(MockMethodInterceptor.java:110)
	at de.zalando.ep.zalenium.proxy.TestingBotRemoteProxy$MockitoMock$1293852747.getTestInformation(Unknown Source)
	at de.zalando.ep.zalenium.proxy.CloudTestingRemoteProxy.lambda$addTestToDashboard$3(CloudTestingRemoteProxy.java:237)
	at java.lang.Thread.run(Thread.java:748)

2020-03-19 19:44:24 INFO   Trying download once again from https://api.testingbot.com/v1/tests/2cf5d115-ca6f-4bc4-bc06-a4fca00836ce 
2020-03-19 19:44:24 WARNING Max instance not specified. Using default = 1 instance 
2020-03-19 19:44:24 SEVERE java.io.FileNotFoundException: https://saucelabs.com/rest/v1//jobs/RANDOM_EXTERNAL_KEY 
java.io.FileNotFoundException: https://saucelabs.com/rest/v1//jobs/RANDOM_EXTERNAL_KEY
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1890)
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1492)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:263)
	at de.zalando.ep.zalenium.util.CommonProxyUtilities.readJSONFromUrl(CommonProxyUtilities.java:43)
	at de.zalando.ep.zalenium.proxy.SauceLabsRemoteProxy.getTestInformation(SauceLabsRemoteProxy.java:107)
	at de.zalando.ep.zalenium.proxy.SauceLabsRemoteProxy$MockitoMock$1783483347.getTestInformation$accessor$fSume9lV(Unknown Source)
	at de.zalando.ep.zalenium.proxy.SauceLabsRemoteProxy$MockitoMock$1783483347$auxiliary$oBuloHYX.call(Unknown Source)
	at org.mockito.internal.invocation.RealMethod$FromCallable$1.call(RealMethod.java:40)
	at org.mockito.internal.invocation.RealMethod$FromBehavior.invoke(RealMethod.java:62)
	at org.mockito.internal.invocation.InterceptedInvocation.callRealMethod(InterceptedInvocation.java:127)
	at org.mockito.internal.stubbing.answers.CallsRealMethods.answer(CallsRealMethods.java:43)
	at org.mockito.Answers.answer(Answers.java:100)
	at org.mockito.internal.handler.MockHandlerImpl.handle(MockHandlerImpl.java:104)
	at org.mockito.internal.handler.NullResultGuardian.handle(NullResultGuardian.java:29)
	at org.mockito.internal.handler.InvocationNotifierHandler.handle(InvocationNotifierHandler.java:35)
	at org.mockito.internal.creation.bytebuddy.MockMethodInterceptor.doIntercept(MockMethodInterceptor.java:63)
	at org.mockito.internal.creation.bytebuddy.MockMethodInterceptor.doIntercept(MockMethodInterceptor.java:49)
	at org.mockito.internal.creation.bytebuddy.MockMethodInterceptor$DispatcherDefaultingToRealMethod.interceptSuperCallable(MockMethodInterceptor.java:110)
	at de.zalando.ep.zalenium.proxy.SauceLabsRemoteProxy$MockitoMock$1783483347.getTestInformation(Unknown Source)
	at de.zalando.ep.zalenium.proxy.CloudTestingRemoteProxy.lambda$addTestToDashboard$3(CloudTestingRemoteProxy.java:237)
	at java.lang.Thread.run(Thread.java:748)

2020-03-19 19:44:24 INFO   Trying download once again from https://saucelabs.com/rest/v1//jobs/RANDOM_EXTERNAL_KEY 
2020-03-19 19:44:24 INFO   Marking the node http://localhost:30000 as down: cannot reach the node for 2 tries 
2020-03-19 19:44:24 INFO   [SL] Account max. concurrency was NOT fetched from https://saucelabs.com/rest/v1/users/ 
2020-03-19 19:44:24 INFO   [BS] Account max. concurrency was NOT fetched from https://www.browserstack.com/automate/plan.json 
2020-03-19 19:44:24 INFO   [TB] Account max. concurrency was NOT fetched from https://api.testingbot.com/v1/user 
2020-03-19 19:44:25 INFO   Unregistering the node http://localhost:40000 because it's been down for 60038 milliseconds 
2020-03-19 19:44:25 INFO   Unregistering the node http://localhost:40000 because it's been down for 65039 milliseconds 
2020-03-19 19:44:25 INFO   Unregistering the node http://localhost:40000 because it's been down for 70044 milliseconds 
2020-03-19 19:44:25 INFO   Registered a node http://localhost:30000 
2020-03-19 19:44:25 INFO   [DS]  Setting up 0 nodes... 
2020-03-19 19:44:25 INFO   0 containers were created, it will take a bit more until all get registered. 
2020-03-19 19:44:25 INFO   Registered a node http://localhost:40000 
2020-03-19 19:44:25 INFO   Registered a node http://localhost:40001 
2020-03-19 19:44:25 INFO   Registered a node http://ondemand.saucelabs.com:80 
2020-03-19 19:44:25 INFO   Registered a node http://hub-cloud.browserstack.com:80 
2020-03-19 19:44:25 INFO   Registered a node http://hub.testingbot.com:80 
2020-03-19 19:44:25 SEVERE java.io.FileNotFoundException: https://saucelabs.com/rest/v1//jobs/testKey 
java.io.FileNotFoundException: https://saucelabs.com/rest/v1//jobs/testKey
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1890)
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1492)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:263)
	at de.zalando.ep.zalenium.util.CommonProxyUtilities.readJSONFromUrl(CommonProxyUtilities.java:43)
	at de.zalando.ep.zalenium.proxy.SauceLabsRemoteProxy.getTestInformation(SauceLabsRemoteProxy.java:107)
	at de.zalando.ep.zalenium.proxy.CloudTestingRemoteProxy.lambda$addTestToDashboard$3(CloudTestingRemoteProxy.java:237)
	at java.lang.Thread.run(Thread.java:748)

2020-03-19 19:44:25 INFO   Trying download once again from https://saucelabs.com/rest/v1//jobs/testKey 
2020-03-19 19:44:25 INFO   Marking the node http://localhost:30000 as down: cannot reach the node for 2 tries 
2020-03-19 19:44:25 INFO   [SL] Account max. concurrency was NOT fetched from https://saucelabs.com/rest/v1/users/ 
2020-03-19 19:44:25 INFO   [BS] Account max. concurrency was NOT fetched from https://www.browserstack.com/automate/plan.json 
2020-03-19 19:44:25 INFO   [TB] Account max. concurrency was NOT fetched from https://api.testingbot.com/v1/user 
2020-03-19 19:44:25 INFO   Registered a node http://localhost:30000 
2020-03-19 19:44:25 INFO   [DS]  Setting up 0 nodes... 
2020-03-19 19:44:25 INFO   0 containers were created, it will take a bit more until all get registered. 
2020-03-19 19:44:25 INFO   Registered a node http://localhost:40000 
2020-03-19 19:44:25 INFO   Registered a node http://localhost:40001 
2020-03-19 19:44:25 INFO   Registered a node http://ondemand.saucelabs.com:80 
2020-03-19 19:44:25 INFO   Registered a node http://hub-cloud.browserstack.com:80 
2020-03-19 19:44:25 INFO   Registered a node http://hub.testingbot.com:80 
2020-03-19 19:44:25 INFO   Unregistering the node http://localhost:40000 because it's been down for 60049 milliseconds 
2020-03-19 19:44:25 INFO   Unregistering the node http://localhost:40000 because it's been down for 65050 milliseconds 
2020-03-19 19:44:26 INFO   Unregistering the node http://localhost:40000 because it's been down for 60021 milliseconds 
2020-03-19 19:44:26 INFO   Unregistering the node http://localhost:40000 because it's been down for 65023 milliseconds 
2020-03-19 19:44:26 INFO   Unregistering the node http://localhost:40000 because it's been down for 70024 milliseconds 
2020-03-19 19:44:26 INFO   Marking the node http://localhost:30000 as down: cannot reach the node for 2 tries 
2020-03-19 19:44:26 INFO   Marking the node http://localhost:30000 as down: cannot reach the node for 2 tries 
2020-03-19 19:44:26 INFO   Unregistering the node http://localhost:40000 because it's been down for 60023 milliseconds 
2020-03-19 19:44:27 INFO   Marking the node http://localhost:30000 as down: cannot reach the node for 2 tries 
2020-03-19 19:44:27 INFO   [SL] Account max. concurrency was NOT fetched from https://saucelabs.com/rest/v1/users/ 
2020-03-19 19:44:27 INFO   [BS] Account max. concurrency was NOT fetched from https://www.browserstack.com/automate/plan.json 
2020-03-19 19:44:27 INFO   [TB] Account max. concurrency was NOT fetched from https://api.testingbot.com/v1/user 
2020-03-19 19:44:27 INFO   Registered a node http://localhost:30000 
2020-03-19 19:44:27 INFO   [DS]  Setting up 0 nodes... 
2020-03-19 19:44:27 INFO   0 containers were created, it will take a bit more until all get registered. 
2020-03-19 19:44:27 INFO   Registered a node http://localhost:40000 
2020-03-19 19:44:27 INFO   Registered a node http://localhost:40001 
2020-03-19 19:44:27 INFO   Registered a node http://ondemand.saucelabs.com:80 
2020-03-19 19:44:27 INFO   Registered a node http://hub-cloud.browserstack.com:80 
2020-03-19 19:44:27 INFO   Registered a node http://hub.testingbot.com:80 
2020-03-19 19:44:27 INFO   Unregistering the node http://localhost:40000 because it's been down for 60035 milliseconds 
2020-03-19 19:44:28 INFO   Trying download once again from https://saucelabs.com/rest/v1//jobs/72e4f8ecf04440fe965faf657864ed52/assets/log.json 
2020-03-19 19:44:28 INFO   Marking the node http://localhost:30000 as down: cannot reach the node for 2 tries 
2020-03-19 19:44:28 INFO   Marking the node http://localhost:30000 as down: cannot reach the node for 2 tries 
2020-03-19 19:44:29 INFO   Registered a node http://localhost:40000 
2020-03-19 19:44:29 INFO   Registered a node http://localhost:40001 
2020-03-19 19:44:29 INFO   Marking the node http://localhost:30000 as down: cannot reach the node for 2 tries 
2020-03-19 19:44:29 SEVERE java.io.IOException: Server returned HTTP response code: 401 for URL: https://api.testingbot.com/v1/tests/2cf5d115-ca6f-4bc4-bc06-a4fca00836ce 
java.io.IOException: Server returned HTTP response code: 401 for URL: https://api.testingbot.com/v1/tests/2cf5d115-ca6f-4bc4-bc06-a4fca00836ce
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1894)
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1492)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:263)
	at de.zalando.ep.zalenium.util.CommonProxyUtilities.readJSONFromUrl(CommonProxyUtilities.java:43)
	at de.zalando.ep.zalenium.proxy.TestingBotRemoteProxy.getTestInformation(TestingBotRemoteProxy.java:93)
	at de.zalando.ep.zalenium.proxy.TestingBotRemoteProxy$MockitoMock$1293852747.getTestInformation$accessor$dF19ulJq(Unknown Source)
	at de.zalando.ep.zalenium.proxy.TestingBotRemoteProxy$MockitoMock$1293852747$auxiliary$2k2Ef5Mz.call(Unknown Source)
	at org.mockito.internal.invocation.RealMethod$FromCallable$1.call(RealMethod.java:40)
	at org.mockito.internal.invocation.RealMethod$FromBehavior.invoke(RealMethod.java:62)
	at org.mockito.internal.invocation.InterceptedInvocation.callRealMethod(InterceptedInvocation.java:127)
	at org.mockito.internal.stubbing.answers.CallsRealMethods.answer(CallsRealMethods.java:43)
	at org.mockito.Answers.answer(Answers.java:100)
	at org.mockito.internal.handler.MockHandlerImpl.handle(MockHandlerImpl.java:104)
	at org.mockito.internal.handler.NullResultGuardian.handle(NullResultGuardian.java:29)
	at org.mockito.internal.handler.InvocationNotifierHandler.handle(InvocationNotifierHandler.java:35)
	at org.mockito.internal.creation.bytebuddy.MockMethodInterceptor.doIntercept(MockMethodInterceptor.java:63)
	at org.mockito.internal.creation.bytebuddy.MockMethodInterceptor.doIntercept(MockMethodInterceptor.java:49)
	at org.mockito.internal.creation.bytebuddy.MockMethodInterceptor$DispatcherDefaultingToRealMethod.interceptSuperCallable(MockMethodInterceptor.java:110)
	at de.zalando.ep.zalenium.proxy.TestingBotRemoteProxy$MockitoMock$1293852747.getTestInformation(Unknown Source)
	at de.zalando.ep.zalenium.proxy.CloudTestingRemoteProxy.lambda$addTestToDashboard$3(CloudTestingRemoteProxy.java:237)
	at java.lang.Thread.run(Thread.java:748)

2020-03-19 19:44:29 INFO   Trying download once again from https://api.testingbot.com/v1/tests/2cf5d115-ca6f-4bc4-bc06-a4fca00836ce 
2020-03-19 19:44:29 INFO   Marking the node http://localhost:30000 as down: cannot reach the node for 2 tries 
2020-03-19 19:44:29 INFO   Registered a node http://localhost:40000 
2020-03-19 19:44:29 INFO   Registered a node http://localhost:40001 
2020-03-19 19:44:30 INFO   Unregistering the node http://localhost:40000 because it's been down for 60038 milliseconds 
2020-03-19 19:44:30 INFO   Unregistering the node http://localhost:40000 because it's been down for 65039 milliseconds 
2020-03-19 19:44:30 INFO   Unregistering the node http://localhost:40000 because it's been down for 70044 milliseconds 
2020-03-19 19:44:30 INFO   Unregistering the node http://localhost:40000 because it's been down for 75054 milliseconds 
2020-03-19 19:44:30 INFO   Marking the node http://localhost:30000 as down: cannot reach the node for 2 tries 
2020-03-19 19:44:30 SEVERE java.io.FileNotFoundException: https://saucelabs.com/rest/v1//jobs/RANDOM_EXTERNAL_KEY 
java.io.FileNotFoundException: https://saucelabs.com/rest/v1//jobs/RANDOM_EXTERNAL_KEY
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1890)
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1492)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:263)
	at de.zalando.ep.zalenium.util.CommonProxyUtilities.readJSONFromUrl(CommonProxyUtilities.java:43)
	at de.zalando.ep.zalenium.proxy.SauceLabsRemoteProxy.getTestInformation(SauceLabsRemoteProxy.java:107)
	at de.zalando.ep.zalenium.proxy.SauceLabsRemoteProxy$MockitoMock$1783483347.getTestInformation$accessor$fSume9lV(Unknown Source)
	at de.zalando.ep.zalenium.proxy.SauceLabsRemoteProxy$MockitoMock$1783483347$auxiliary$oBuloHYX.call(Unknown Source)
	at org.mockito.internal.invocation.RealMethod$FromCallable$1.call(RealMethod.java:40)
	at org.mockito.internal.invocation.RealMethod$FromBehavior.invoke(RealMethod.java:62)
	at org.mockito.internal.invocation.InterceptedInvocation.callRealMethod(InterceptedInvocation.java:127)
	at org.mockito.internal.stubbing.answers.CallsRealMethods.answer(CallsRealMethods.java:43)
	at org.mockito.Answers.answer(Answers.java:100)
	at org.mockito.internal.handler.MockHandlerImpl.handle(MockHandlerImpl.java:104)
	at org.mockito.internal.handler.NullResultGuardian.handle(NullResultGuardian.java:29)
	at org.mockito.internal.handler.InvocationNotifierHandler.handle(InvocationNotifierHandler.java:35)
	at org.mockito.internal.creation.bytebuddy.MockMethodInterceptor.doIntercept(MockMethodInterceptor.java:63)
	at org.mockito.internal.creation.bytebuddy.MockMethodInterceptor.doIntercept(MockMethodInterceptor.java:49)
	at org.mockito.internal.creation.bytebuddy.MockMethodInterceptor$DispatcherDefaultingToRealMethod.interceptSuperCallable(MockMethodInterceptor.java:110)
	at de.zalando.ep.zalenium.proxy.SauceLabsRemoteProxy$MockitoMock$1783483347.getTestInformation(Unknown Source)
	at de.zalando.ep.zalenium.proxy.CloudTestingRemoteProxy.lambda$addTestToDashboard$3(CloudTestingRemoteProxy.java:237)
	at java.lang.Thread.run(Thread.java:748)

2020-03-19 19:44:30 INFO   Trying download once again from https://saucelabs.com/rest/v1//jobs/RANDOM_EXTERNAL_KEY 
2020-03-19 19:44:30 INFO   Registered a node http://localhost:40000 
2020-03-19 19:44:30 INFO   Registered a node http://localhost:40001 
2020-03-19 19:44:30 INFO   Unregistering the node http://localhost:40000 because it's been down for 60049 milliseconds 
2020-03-19 19:44:30 INFO   Unregistering the node http://localhost:40000 because it's been down for 65050 milliseconds 
2020-03-19 19:44:30 INFO   Unregistering the node http://localhost:40000 because it's been down for 70055 milliseconds 
2020-03-19 19:44:31 INFO   Unregistering the node http://localhost:40000 because it's been down for 60021 milliseconds 
2020-03-19 19:44:31 INFO   Unregistering the node http://localhost:40000 because it's been down for 65023 milliseconds 
2020-03-19 19:44:31 INFO   Unregistering the node http://localhost:40000 because it's been down for 70024 milliseconds 
2020-03-19 19:44:31 INFO   Unregistering the node http://localhost:40000 because it's been down for 75027 milliseconds 
2020-03-19 19:44:31 INFO   Registered a node http://localhost:40000 
2020-03-19 19:44:31 INFO   Registered a node http://localhost:40001 
2020-03-19 19:44:31 SEVERE java.io.FileNotFoundException: https://saucelabs.com/rest/v1//jobs/testKey 
java.io.FileNotFoundException: https://saucelabs.com/rest/v1//jobs/testKey
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1890)
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1492)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:263)
	at de.zalando.ep.zalenium.util.CommonProxyUtilities.readJSONFromUrl(CommonProxyUtilities.java:43)
	at de.zalando.ep.zalenium.proxy.SauceLabsRemoteProxy.getTestInformation(SauceLabsRemoteProxy.java:107)
	at de.zalando.ep.zalenium.proxy.CloudTestingRemoteProxy.lambda$addTestToDashboard$3(CloudTestingRemoteProxy.java:237)
	at java.lang.Thread.run(Thread.java:748)

2020-03-19 19:44:31 INFO   Trying download once again from https://saucelabs.com/rest/v1//jobs/testKey 
2020-03-19 19:44:31 INFO   Marking the node http://localhost:30000 as down: cannot reach the node for 2 tries 
2020-03-19 19:44:31 INFO   Registered a node http://localhost:40000 
2020-03-19 19:44:31 INFO   Registered a node http://localhost:40001 
2020-03-19 19:44:31 INFO   Unregistering the node http://localhost:40000 because it's been down for 60023 milliseconds 
2020-03-19 19:44:31 INFO   Unregistering the node http://localhost:40000 because it's been down for 65025 milliseconds 
2020-03-19 19:44:32 INFO   Marking the node http://localhost:30000 as down: cannot reach the node for 2 tries 
2020-03-19 19:44:32 INFO   Registered a node http://localhost:40000 
2020-03-19 19:44:32 INFO   Registered a node http://localhost:40001 
2020-03-19 19:44:32 INFO   Unregistering the node http://localhost:40000 because it's been down for 60029 milliseconds 
2020-03-19 19:44:32 INFO   Unregistering the node http://localhost:40000 because it's been down for 60035 milliseconds 
2020-03-19 19:44:32 INFO   Unregistering the node http://localhost:40000 because it's been down for 65045 milliseconds 
2020-03-19 19:44:32 INFO   Registered a node http://localhost:40000 
2020-03-19 19:44:32 INFO   Registered a node http://localhost:40001 
2020-03-19 19:44:32 INFO   Marking the node http://localhost:30000 as down: cannot reach the node for 2 tries 
2020-03-19 19:44:33 INFO   Registered a node http://localhost:40000 
2020-03-19 19:44:33 INFO   Registered a node http://localhost:40001 
2020-03-19 19:44:33 INFO   Trying download once again from https://saucelabs.com/rest/v1//jobs/72e4f8ecf04440fe965faf657864ed52/assets/log.json 
2020-03-19 19:44:33 INFO   Registered a node http://localhost:40000 
2020-03-19 19:44:33 INFO   Registered a node http://localhost:40001 
2020-03-19 19:44:34 WARNING java.lang.NumberFormatException: For input string: "Not-A-Number" 
java.lang.NumberFormatException: For input string: "Not-A-Number"
	at java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
	at java.lang.Integer.parseInt(Integer.java:580)
	at java.lang.Integer.parseInt(Integer.java:615)
	at de.zalando.ep.zalenium.dashboard.Dashboard.synchronizeExecutedTestsValues(Dashboard.java:172)
	at de.zalando.ep.zalenium.dashboard.Dashboard.updateDashboard(Dashboard.java:81)
	at de.zalando.ep.zalenium.dashboard.DashboardTest.nonNumberContentsIgnored(DashboardTest.java:84)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:48)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.apache.maven.surefire.junitcore.JUnitCore.run(JUnitCore.java:55)
	at org.apache.maven.surefire.junitcore.JUnitCoreWrapper.createRequestAndRun(JUnitCoreWrapper.java:137)
	at org.apache.maven.surefire.junitcore.JUnitCoreWrapper.executeEager(JUnitCoreWrapper.java:107)
	at org.apache.maven.surefire.junitcore.JUnitCoreWrapper.execute(JUnitCoreWrapper.java:83)
	at org.apache.maven.surefire.junitcore.JUnitCoreWrapper.execute(JUnitCoreWrapper.java:75)
	at org.apache.maven.surefire.junitcore.JUnitCoreProvider.invoke(JUnitCoreProvider.java:158)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:379)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:340)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:125)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:413)

